<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta name="description" content="Network2Canvas - connecting library terms using shared attributes and visualizing them on a 2-D SVG Canvas." />
	<meta name="keywords" content="Avi Ma'ayan maayan laboratory mount sinai school of medicine new york mssm network2canvas n2c network 2 canvas" />
    <meta name="author" content="Christopher Tan" />
	<title>Network2Canvas</title>

	<link rel="stylesheet" type="text/css" href='../Resources/visualizer5.css' />


	<script type='text/javascript' src='../Resources/d3.v2.min.js'></script>
	<script type="text/javascript" src='../Resources/html5slider.js'></script>
	<script type='text/javascript' src='../Resources/offlineVisualizer.js'></script>
</head>

<body>
	<div id="externalwrapper">
		<div id="holdElements">
		
		
			<div id="topbar">
				<span class= "linker" style="font-weight:700">Offline N2C</span>

			</div>
			
			
			<div id="header" style="font-size:20px; font-family:arial; font-weight:700">Network2Canvas: Network Visualization on a Canvas with Enrichment Analysis</div>
			<div id="menu" class="outputMenu">
					<span class="tab" id="outputTab1" onclick="displayOutput(1)">Info</span>
					<span class="tab" id="outputTab2" onclick="displayOutput(2)" style="font-weight:400; background-color: #CCC;">Analysis Output</span>
			</div>

			<div id="outputDisplay1">
				<!--<div id="SD1"><span id="SD1">Node Information from GMT</span><title>Click a spot on the canvas to view Node Information</title></div>-->

				<div id="nodeInformation">
					<div class="displayTitle">Term Name:</div>
					<div id="nodeName" style="padding-top:5px;padding-bottom:10px;"></div>
				</div>
				
				<span id="additTitle" style="font-size:11px">Additional Information:</span>
				<div id="infoContainer">
					<div id="additionalInfo"></div>
				</div>
				<div style="position:absolute; top:525px; text-align:center; width:191px">
				<input type="button" onClick="document.getElementById('genes').value = document.getElementById('additionalInfo').innerHTML.split(', ').join('\n'); document.getElementById('genes').onfocus=''" value="Add to Input List" />
				</div>
			</div>
		<div id="outputDisplay2" class="GSEA" style="display:none; border-bottom: 1px solid black;">
			<div id="frozenTable" style="padding-bottom:10px;">
				<table class="contain" style="position:absolute;top:20px;">
				<th>Term</th>
				<th>P-Value</th>
			</table>
			</div>
			<div id="enrichmentResults" style="padding-top:10px; position:absolute; top:50px;">

			</div>
		
		</div>
<!--
			<div id="termPanel" style="position:absolute; top:250px; left:10px; border: 1px solid black;">
				<form>
					Find Elements of Term:<br/>
					<textarea id="elementTerm" onfocus="clearTextArea(this.form.genes)" onBlur="getElements(this.form.elementTerm.value);">Insert Term</textarea>
				</form>
			</div>
-->

			<div id="sidePanel">
				<div id="menu">
					<span class="tab" id="tab1" onclick="displayTab(1)">Analysis</span>
					<span class="tab" id="tab2" onclick="displayTab(2)" style="font-weight:400; background-color: #CCC;">Canvas Options</span>
				</div>	
				<form id="form1"><br />
					<p><b>Gene List Enrichment Analysis</b></p>
					Choose Color to Highlight Terms:
					<br /><br />
						<div id="hexcode_gene">
						</div>
						<br />
					
					Gene List:<br /><br />

							<div class="center">
								<textarea id="genes" onfocus="clearTextArea(this.form.genes)" name="genes">Enter list of gene names here</textarea>
							</div>
							<div class="center">
								<input id = "submitButton" type ="button" name="button" value="Submit" selected="true" onClick="geneFill(G_VAR.nodes, this.form.genes.value, G_VAR.indicatorColor, G_VAR.infoDict, G_VAR.infos);"/>
								<input  type="button" name="button" value="Clear" onclick="indicateClear(G_VAR.nodes);"/><br />
								<input id ="exampleButton" type="button" name="button" value="Example" onclick="exampleRandom(G_VAR.infos, G_VAR.randomArray, 10);" />
							</div>

				</form>
				<form id="form2">
					<br />
					<br />


					<span class="canvasOptions">Center Canvas:</span>
				<div class="right">
					<button type="button" onclick="centerCanvas(G_VAR.nodes, G_VAR.canvasSize)">Center</button></div>
				<br />
				<br />
	

				<span class="canvasOptions">Choose Canvas Color:</span> <div class="right"><br />

				<div id="wheelContain"></div>
				</div>
				<br />	
				<br />
				<br />

				<span class="canvasOptions">Color Scale: </span><em id="colorScale">1	</em>
				<div class="right">
					<input type="range" min="0" max="1" value="1" step=".001" name="scale" id="range_colorScale" onMouseup="scaleColor(G_VAR.nodes, G_VAR.avgWeight, this.form.scale.value, G_VAR.canvasRGB, G_VAR.canvasSize);" onChange="seeScale(this.form.scale.value);"></input><br />
					<input type="button" name="button" value="Reset Scale" onclick="resetColorScale(G_VAR.nodes, G_VAR.canvasRGB, G_VAR.canvasSize);"/>
				</div><br />
				<br />


				</form>
				<br />
				<br />
				<div id="findNode" style="border-top: 1px solid black;">
				
				<div id="menu">
					<b style="display:inline-block; padding-right:10px;">Find by:</b>
					<span class="tab" id="tabFind1" onclick="displayFind(1)">Name</span>
					<span class="tab" id="tabFind2" onclick="displayFind(2)" style="font-weight:400; background-color: #CCC;">Element</span>
				</div>	
					<form id="formFind1">
					<p><b style="padding-right:5px">Find Terms by Name: </b><br /><br />
							<textarea id="nodes" names="nodes" cols="10" rows="3" onfocus="clearTextArea(this.form.nodes)">Insert Term List (e.g.,          CDK1            CDK2            CDK3)</textarea>
							
							
							<input type="button" onclick="fill(G_VAR.nodes, this.form.nodes.value, G_VAR.indicatorColor , G_VAR.width);" value="Find"/>
						</p>
					</form>
					<form id="formFind2" style="display:none;">
					<p><b style="padding-right:5px">Find Terms by Element(s): </b><br /><br />
							<textarea id="nodes" names="nodes" cols="10" rows="3" onfocus="clearTextArea(this.form.nodes)">Insert Element List (e.g.,          CDK1            CDK2            CDK3)</textarea>
							
							<input type="button" onclick="fillElement(G_VAR.nodes, this.form.nodes.value, G_VAR.indicatorColor , G_VAR.width);" value="Find"/>
							
						</p>
					</form>
				</div>
				<hr />
				<br />
				<a href="#" id="download" onclick="downloadLink();">Download Current View in SVG<br/>(Pathway to Adobe Illustrator)</a>

				<br />
				
				</div> 


			<div id="holdSVG">

				<div id="svgWrapper">
					<form id="selectSVG">

						<select id="selectCanvas" onChange="getGMT(this.form.selectCanvas.selectedIndex);getJSON(this.form.selectCanvas.selectedIndex, G_VAR.canvasRGB, G_VAR.indicatorColor);" disabled = "true">
				
						</select>
						</form>

				<div id="svgContainer"></div>
				<div id="chartContainer"></div>
				<div id="pvalueContainer"></div>
				<form id="choiceView" style="padding-top:10px">
						<input type ="radio" id="mainCanvasSelector" name="a" checked="checked" onClick="selectAlternateView()">Main View</input>
						<input type ="radio" id="networkView" name = "a" disabled="true" onClick="selectAlternateView()">Network View</input>
						<input type ="radio" id="pvalueCanvas" name = "a" disabled="true" onClick="selectAlternateView()">P-Value View</input>
				</form>

				</div>


				
			</div>



			<div id="manhattanElement">
				<div id="selectionDisplay3">

				
				</div>
			</div>

			</div>
		</div>
		<div id="footer">
			(c) 2012 <a href="http://www.mssm.edu/labs/maayan">Ma'ayan Laboratory</a> - <a href="http://www.mssm.edu">Mount Sinai School of Medicine</a>
		</div>
	


</body>


<script type="text/javascript">
	colorWheel(10, "div#wheelContain", "G_VAR.canvasRGB", [0, 255, 255]);
	colorWheel(10, "div#hexcode_gene", "G_VAR.indicatorColor", [255, 255, 255]);
	
d3.select('select#selectCanvas').append('option').text('Input/canvas2_matrix.txt');



var json = {"texts": ["MIRACH", "CloudPhylo", "HGVbaseG2P", "UniProt-DAAC", "methylFlow", "SoyBase", "virtualArray", "FIGfams", "GARLIC", "edgeRun", "PseudoBase++", "U-Compare bio-event meta-service", "InParanoid", "GibbsCluster", "miRTar2GO", "ODORactor", "SylArray", "SEARCHGTr", "WebRASP", "SuperDrug", "Bandage", "DAMPD", "CloVR", "TranspoGene", "RIPPER", "repo", "HOMER", "QuartetS", "doRiNA", "FRED", "PRODIGY", "GeMSTONE", "stpm", "PubMed-EX", "JBioWH", "ELOPER", "MESMER", "H-PoP and H-PoPG", "GenoExp", "SPiCE", "bwtool", "CORUM", "AutoFACT", "'PACLIMS'", "MAHMI", "PharmMapper", "ICEberg", "CRISPRmap", "Hembase", "VSEAMS", "CORRIE", "Chimera", "BayesPI2", "BATS", "PIGSPro", "GOstat", "GigaTON", "AQUA-DUCT", "MBGD", "PsRobot", "EnzymeDetector", "ConSurf", "ARMADA", "3DSwap", "CellProfiler", "DensityMap", "Rgb", "SuiteMSA", "GRAPE", "H-BLAST", "GppFst", "COMA", "Pfold", "TMRPres2D", "REST", "Imogene", "TIPP", "OmniMapFree", "TAP-k", "Kismeth", "SigReannot-mart", "YODA", "TRRD", "BEN", "PINALOG", "CLASS2", "GRENDEL", "limmaGUI", "CBESW", "Bio3D-web", "Age-Phenome", "MRMPlus", "MeT-DB", "MAGNA++", "EzCatDB", "DepthTools", "SparkBLAST", "ANDY", "ChloroSSRdb", "ATIVS", "GeNESiS", "ORDB", "PDB-related", "Phenol-Explorer 3.0", "KOBAS", "Adapt-Mix", "MGDB", "GenColors", "DBSecSys", "FARME", "JAIL", "PyMOL", "ISCB-Asia", "F2C2", "GNAT", "WGBSSuite", "PhosphoChain", "Phage Eco-Locator", "GE-mini", "ProbMetab", "DANN", "transFold", "EcoliWiki", "LC/MS", "EssOilDB", "MRPrimerV", "Locomotif", "IUPred", "UrQt", "euGenes", "WebCARMA", "Genovar", "PPUS", "CDSbank", "SPARSE", "MapLinker", "ICMA", "GXD", "Muxstep", "PPT-DB", "SensA", "PSMIX", "PanViz", "TreeSAAP", "ZINCPharmer", "HapMap", "StemCellNet", "CompaRNA", "PHI-base", "HapZipper", "INSYGHT", "M-Coffee", "METscout", "SLMSuite", "OligoWiz", "EGNAS", "ASEdb", "LEGER", "VizRank", "Figaro", "G-NEST", "MT-Toolbox", "DNAism", "GeneMark", "Vacceed", "PiDNA", "MS-DOCK", "BamView", "MOM", "KvSNP", "BrainTrap", "FunTree", "EKPD", "LIBRA", "IgBLAST", "EMPRES-i", "varDB", "ClanTox", "TissueNet", "Jalview", "LncRNA-ID", "MAP-RSeq", "AD-LIBS", "BOMP", "pplacer", "MAQC", "ReLiance", "Bellerophontes", "NFPscanner", "gpDB", "PinSnps", "mFASD", "SNPStats", "DASHR", "GeIST", "LightAssembler", "CheNER", "TESE", "MetaSpark", "MutaBind", "hmChIP", "DichroMatch", "GLOOME", "GASSST", "PatternLab", "HmtDB", "GnpIS", "COMPADRE", "GlimmerM", "PPfold", "PESDserv", "InterferenceAnalyzer", "NMR", "GOAL", "ISbrowser", "URec", "TiMEx", "IFIM", "EasyCluster2", "StellaBase", "IMA", "Bioclipse-R", "NTAP", "M4T", "BranchClust", "CONREAL", "S3QL", "PhyTB", "SNP500Cancer", "ONTO-ToolKit", "ArrayMaker", "BioSilico", "library-CCDB", "MOST", "DroID", "Insyght", "MitoAge", "Primerize", "IBiSS", "Apollo", "SODa", "MoRFpred", "SeqLib", "NetPhosYeast", "PREDITOR", "IPD", "MASiVE", "FuMa", "SilkPathDB", "Rust-Bio", "GARD", "WholeCellSimDB", "RASMOT-3D PRO", "MorusDB", "ConoDictor", "PLAZA", "MiDAS", "BOND", "DBAli", "PolyMarker", "ParticleCall", "ImmuneDB", "ALOHOMORA", "LIQUID", "BreCAN-DB", "M-AMST", "PopGeV", "CBDB", "SimArray", "IntEnz", "SpliceSeq", "MulRF", "Lynx", "SplAdder", "RNAstructure", "SOAP-based", "PEDB", "EpiGenome", "jHeatmap", "PAVIS", "RVFam", "NOVOPlasty", "GR-Align", "4SALE", "ChromoZoom", "DODO", "G-quadruplexes in viruses", "Genes2FANs", "MetaMIS", "eF-site and PDBjViewer", "QUADrATiC", "LISE", "flowAI", "deBGA", "SeqSIMLA", "PharmGKB", "lncRNAdb", "Fuse", "CMKb", "ARGs-OAP", "FluoRender", "EcoliNet", "ProDiGe", "Omicseq", "SNPsea", "OEFinder", "BAYSIC", "CAGEr", "Pasha", "MindSeer", "3DTF", "BioC-compatible", "GO2MSIG", "HECTAR", "WFES", "ADaCGH2", "MemGen", "MetaPred2CS", "DRUMS", "ArchSchema", "drexplorer", "TIMBAL", "RevTrans", "GPMDB", "BioCatalogue", "ASTALAVISTA", "YeastGenome", "bioNerDS", "Jflow", "mpMoRFsDB", "POLYVIEW-3D", "CoD", "DNAshapeR", "PAL2NAL", "OntoQuery", "HAL", "Goldilocks", "NxTrim", "solarius", "PC-TraFF", "JACUSA", "ProteomeScout", "DrumPID", "DiProGB", "XMSF", "ampliMethProfiler", "SSEP", "T-RMSD", "NRPSsp", "MRCQuant-", "PivotViewer", "Alloscore", "R3D-2-MSA", "FireDB", "ACFIS", "OrthoClusterDB", "Rcpi", "BreakFusion", "QDMR", "ESE", "wwPDB", "BioLINK", "PCIT", "TopHat", "AUGUSTUS", "MapMyFlu", "Pseudofam", "SysGenSIM", "SMRT", "DIANA-microT", "CanGEM", "RCytoscape", "miRTrail", "DOOR", "DIPSBC", "gkmSVM", "UniProt-GO", "Bison", "HomSI", "TALENoffer", "NuST", "ISOL@", "ROCker", "RareVariantVis", "Phospho.ELM", "flowClust", "FlyExpress", "CASPER", "VBASE2", "RiboFR-Seq", "ADGO", "seqlm", "XQTav", "Poretools", "ImmunoNodes", "ProtSA", "CABS-fold", "MiMI", "OmegaPlus", "ModBase", "RNALogo", "CrossTope", "HHalign-Kbest", "PASS-bis", "keeSeek", "CPPsite", "SYNBADm", "SVM", "FPSAC", "SCOPA and META-SCOPA", "LoRDEC", "GlycoProfileAssigner", "CaPSID", "LRSSL", "RGED", "svaseq", "repDNA", "OBO", "SAMSA", "CellTracker", "iMotifs", "HOR", "PeakAnalyzer", "DAnCER", "MetaBioME", "HITS-CLIP", "AREsite2", "MotifVoter", "GEB", "SSBD", "PGAP", "CymeR", "MAGIIC-PRO", "systemsDock", "PEAT", "FreeContact", "SNP-seek", "SUBAcon", "GeConT", "BRAKER1", "DOSim", "FungiDB", "PAA", "MitoBreak", "Allelome.PRO", "SVDetect", "GPViz", "LW-FQZip 2", "DoseSim", "myPhyloDB", "miRiadne", "BRAD", "ontoCAT", "ChemiRs", "ChiTaRS-3.1\u2014the", "GenomeCRISPR", "PopIns", "JAR3D", "GPLEXUS", "OGDD", "miRCancer", "dbCAN", "TieDIE", "Manta", "INSECT", "MIAMExpress", "InterEvol", "PACdb", "Largenet2", "DrugQuest", "VarDetect", "NetCooperate", "PSI-BLAST", "ExpressYourself", "PRISM", "ENIGMA-Viewer", "Chembench", "canSAR", "SRV", "SBMLmod", "Tabhu", "GermOnline", "InterPro", "H-InvDB in 2013", "Seq2Ref", "KINARI-Web", "NetMHC-3.0", "CoCoNUT", "BioInfo3D", "DBAASP", "Fit3D", "AGE", "SpliceInfo", "libFLASM", "BioMoby", "JASPAR", "TIP", "VarMod", "LDsplit", "PERGOLA", "SporeWeb", "metaX", "sRNAMap", "iAlign", "CARGO", "GPI", "Cgaln", "JCoDA", "CNVRuler", "EHFPI", "myGrid", "BGT", "GeNOSA", "IBIS", "Fast-SL", "HTS-DB", "ASD", "MetaboAnalyst", "ASBench", "FastPros", "FMM", "RADB", "TFmodeller", "ToppCluster", "Quikr", "InnateDB", "AthaMap-assisted", "NEAT", "GPU-BLAST", "MCBIOS", "RightField", "iPTM-mLys", "AMDORAP", "CoVDB", "M-path", "svviz", "SNPsyn", "PSSweb", "MP", "PhysBinder", "DGIdb", "ProFET", "RLDA", "SARS-CoV", "JDet", "MODalign", "GLINT", "MIM", "simuPOP", "FACTERA", "BTW", "MicRhoDE", "proABC", "BGMUT", "TreeFam", "ADMIXTURE", "VTBuilder", "PRED-GPCR", "ToppGene", "IntegratedMRF", "pegas", "VISA", "InterProScan", "FNTM", "MUGBAS", "DevMouse", "Sircah", "MUSI", "PASSion", "PGMapper", "SOPRA", "rMAPS", "ArrayInitiative", "DEPTH", "PsyGeNET", "JVirGel", "Seqping", "hotspot", "GNU", "ARM", "CytoSEED", "PubServer", "RiceSRTFDB", "WSxM", "T-Coffee", "Resmap", "CANOES", "SynTView", "MatrixDB", "SUBA4", "SCFIA", "ViralPhos", "Drug-Path", "SBSPKS", "FABSIM", "Tedna", "Structator", "LiverTox", "TagDust2", "EcoGene", "BGX", "DUX4", "DNMAD", "VCF.Filter", "DisGeNET-RDF", "SLiMPrints", "e-RNA", "BioSmalltalk", "PhenomeScape", "Ulla", "dupRadar", "FamAgg", "RNAshapes", "DroPhEA", "CorrelaGenes", "Eoulsan", "PubTator", "PHYML", "GeCo++", "CFMDS", "RNAJunction", "DANCE", "SCMHBP", "RUbioSeq", "COTRASIF", "CAMP", "PEPPI", "HHsvm", "GeneSeeker", "IntSide", "Heatmap3", "3DSNP", "SBtab", "SEWAL", "BioPartsDB", "OligoCalc", "SWAP-Assembler", "WaveletQuant", "Pydna", "DisGeNET", "PyHLA", "Mitome", "Tricross", "ESBTL", "PhosphoGRID", "ELISA-BASE", "MASPECTRAS", "MAVisto", "Voronto", "SPACE", "deacetylation-dependentFVII-gene", "CMDS", "Dfam", "Allerdictor", "Coala", "DASMIweb", "Mendel-GPU", "HUGE", "GeLL", "LS-GKM", "Biomine", "ProTarget", "customProDB", "PTMcode", "EURISCO", "PVS", "TSSV", "M3", "MetaSV", "OrthoInspector", "MACOED", "COBRA", "AgIn", "PyPanda", "SDPMOD", "Kiwi", "MetaCrop", "cn.MOPS", "PASMet", "RGD", "AutoBind", "SpirPro", "PFASUM", "PhaseTank", "ScaffMatch", "DrugE-Rank", "PDBWiki", "JSBML", "MeTaDoR", "Metavir", "PhenoM", "MOSBIE", "PathoYeastract", "dbWFA", "DamID", "topPTM", "Integrated Bio-Search", "SITECON", "GPCRRD", "Pln24NT", "toxoMine", "PlantPAN", "IPD-MHC 2.0", "PhyloDet", "LipocalinPred", "PathPred", "GPCRdb", "Super", "eGenVar", "VARIANT", "GLASS", "WellReader", "FlyBase", "Sub-HMMs", "QmRLFS-finder", "PRADA", "ChemmineR", "libSBML", "MAGIC-web", "tqDist", "SKINK", "HOTA", "ChIP-Enrich", "pdb-care PDB CArbohydrate REsidue check", "UniCon3D", "FFAS-3D", "CyanOmics", "PhenoScanner", "E-zyme", "POSA", "DanQ", "GermlncRNA", "Structure-PPi", "PEDANT", "NGSANE", "PDBFlex", "R-Coffee", "PubMeth", "SIMAT", "AraPheno", "HyperBrowser", "MetaFIND", "Re-searcher", "RACER", "FDR", "Spi-1/PU.1", "Gowinda", "QuadBase2", "NESbase", "FirebrowseR", "DopaNet", "FUN-L", "Gener", "HotSpot", "OperonDB", "FoldNucleus", "RawGeno", "PyMS", "Sifter-T", "pseudoMap", "LS-SNP", "DOMIRE", "SWAKK", "Biomartr", "MMM", "DiNuP", "LOLA", "DIYABC", "Clearcut", "Gene-oriented ortholog database", "MEGADOCK", "PROF_", "PathMAPA", "R2C", "ISMB/ECCB", "Bio3D", "PCDDB", "BicOverlapper", "AHD2.0", "CPC2", "Populous", "Bio-TDS", "Kablammo", "ChemProt", "SPdb", "SAMSCOPE", "DACTAL", "GeneDig", "ABIF", "OntoCAT", "TAPIR", "inTB", "MICAN", "FOLD-EM", "GWAS", "ODIN", "CrossChip", "CODEX", "QuasR", "VacSol", "FetalQuant", "miRDeep*", "ECMIS", "TADB", "Breaking-Cas\u2014interactive", "Mtreemix", "phyloSkeleton", "TriageTools", "ExomeAI", "IDBA-UD", "ENGINES", "LMAP", "MSDmotif", "MODEST", "ARA-PEPs", "PlasmoDB", "BALL", "SPIKE", "BioConductor", "NaviSE", "GLIDERS", "PosiGene", "FLOR-ID", "Tools4miRs", "PPDB", "TarO", "PALADIN", "MultiQC", "IMG-ABC", "Sigma-RF", "nucleR", "Decombinator", "PhyloSim", "MicroRNA-target", "SBRML", "BactImAS", "METIS", "SUPER-FOCUS", "miR2GO", "MiRror", "CycleFreeFlux", "RNAmute", "MSAProbs-MPI", "Biospark", "NEP", "euL1db", "ChIA-PET2", "NAT2", "GonadSAGE", "HCS-Analyzer", "MaM", "BION", "BDML", "Mirsynergy", "BioPortal", "PLEXdb", "AMiGA", "CicArMiSatDB", "BEL", "Scooby-domain", "ChEpiMod", "ALGGEN", "Hemolytik", "GeStoDifferent", "jSplice", "Rfold", "HIPPIE", "ICD-9-CM", "EK3D", "MBRole", "StochHMM", "BAGEL", "MITE", "STAR3D", "SRide", "flowFit", "SA-SSR", "DEOP", "SSAKE", "GATA", "MetNet", "jvenn", "PhiSpy", "MEMBPLUGIN", "MetaRoute", "TriatoKey", "phangorn", "Methylation-Specific MLPA MS-MLPA", "BOCTOPUS", "SnoReport", "Meta-aligner", "PDB-Explorer", "TESD", "Paratome", "ORIO", "OrthoSelect", "MSMAD", "ESTExplorer", "FluxCTTX", "ANCHOR", "TandTRAQ", "DIVE", "Rail-dbGaP", "CellCircuits", "TANDEM", "IMG", "SSB", "SMURFLite", "gsrc", "SLOPE", "Araport", "ShatterProof", "TOPDB", "DIALIGN-T", "IRMa", "mlegp", "InFiRe", "ScalaBLAST", "CADgene", "TcruziDB", "RCPedia", "CAMUR", "RNAcentral", "Argo", "MAGIC-SPP", "NSMAP", "JIGSAW", "PTM-SD", "compEpiTools", "Selecton", "VISTA", "IMB", "RamiGO", "FuncPatch", "SELPHI", "NGL", "ISHAPE", "BitTorious", "BETASEQ", "SUPERFICIAL", "TEMP", "MEGARes", "Goulphar", "Treephyler", "DACE", "miRSel", "PGSB", "ESA-UbiSite", "Dalliance", "SDEAP", "RESCUE-ESE", "RxnSim", "SSuMMo", "MSIdV", "sBWT", "Arena3D", "PhenoDigm", "MiningABs", "NVT", "INTERSNP", "MethylViewer", "LuciPHOr2", "Data2Dynamics", "Open TG-GATEs", "CWRML", "PepX", "Voroprot", "Bioclipse", "SubpathwayMiner", "Furby", "PDB-wide collection of binding data", "SwiftLink", "TIMMA-R", "SVGenes", "img2net", "Cross-React", "MetaQuant", "TarPred", "MethylAid", "AVPpred", "MetaPlotR", "HiPub", "DILIMOT", "MGEScan", "AltAVisT", "GWASdb", "PseudoViewer3", "Selenoprofiles", "BIG", "DMLE+", "PID", "BIO", "SVPV", "STAMS", "ALTRE", "NaviCell", "Aro", "rVISTA", "pre-miRNA", "Mirin", "StatAlign", "GLAD", "kmer-SVM", "miRvestigator", "DP-Bind", "antaRNA", "NIDDK", "RCI", "SPRING", "InteroPORC", "ProPortal", "DPTF", "FragGeneScan", "BiologicalNetworks", "ViralZone", "SHAPE-directed", "Rspider", "PRESS", "PharmaGist", "LowMACA", "CYPSI", "CRISPRFinder", "TRAPP", "Transterm", "DIGIT", "gEVE", "MYCbase", "MSeasy", "PhenoGO", "RuleMonkey", "SuperFly", "CellMontage", "DEsubs", "DECODE", "PREDDIMER", "STEM", "BioC-BioGRID", "Vir-Mir db", "genome-scaleArabidopsis-Pseudomonas", "PhyloBayes", "PAComplex", "EBSeq-HMM", "PubMed", "HiTRACE-Web", "WAViS", "BioSurfDB", "OMA", "PSI-Search", "GREENC", "UniMaP", "MolAxis", "LinkProt", "RegenBase", "PADRE", "PhyloGene", "ConPlex", "CRISPI", "ConBind", "MIDAS", "CMSA", "Genomicus", "WIWS", "SPLATCHE2", "GBS-SNP-CROP", "LOTUS-DB", "DIANA-LncBase", "opm", "BioIMAX", "SoloDel", "Beegle", "TargetDB", "APRICOT", "fqtools", "T3SEdb", "Waggawagga", "IntAct", "RNAontheBENCH", "miRTar", "HaploPainter", "dictyExpress", "CisMols", "incaRNAfbinv", "MUMAL2", "HRaP", "caspo", "Scipio", "HiTRACE", "PDZ", "CIG-P", "PathCards", "MetaMapp", "BioServices", "HaploGrep", "pknotsRG", "A-DaGO-Fun", "BioXpress", "UProC", "BEAT", "PurBayes", "FaSD-somatic", "Phytozome", "HubAlign", "AMIC@", "iFoldRNA", "Cloudgene", "PODB", "NGSUtils", "EpiFire", "CRASP", "DMINDA", "FastNCA", "Lines-of-Sight", "NCBI", "EPD", "QTLNetwork", "SIMoNe", "H2DB", "permGPU", "SureChEMBL", "S2S", "PETcofold", "PRINCESS", "NaviCom", "XSTREAM", "GUILDify", "miRSponge", "GlycomeAtlas", "INTEGRALL", "Decon2LS", "AmiGO", "Boiler", "REEF", "XTMS", "KLIFS", "Genefu", "CanSNPer", "YeTFaSCo", "PS2", "MANORAA", "PedVizApi", "CSO", "FIDEA", "MAGIA2", "AtmiRNET", "ReviSTER", "CP-CHARM", "MixChIP", "OD-seq", "SynBlast", "GWAlpha", "Intrinsic flexibility of B-DNA", "CINOEDV", "BioQ", "SynLethDB", "DARS-RNP and QUASI-RNP", "SnoopCGH", "Kaviar", "ecoPrimers", "Ariadne", "MESSI", "SEA", "A-MADMAN", "SPSmart", "PyroHMMsnp", "GSA-Lightning", "CR-EST", "IKAP", "CAR", "CSSSCL", "SolexaQA", "FindGeo", "Cd-hit", "CATdb", "FARNA", "UEA", "ERGC", "BioPAX", "PhyloDetect", "HCGene", "TAED", "SAMtools", "PhyloWidget", "EVEX", "SNAD", "ProbeLynx", "InterEvDock", "Cinteny", "flowCL", "ogaraK", "LimTox", "TASSEL", "TRAPLINE", "BRAT-nova", "PaVESy", "GPS-SUMO", "Knime4Bio", "corRna", "SNVer", "CruzDB", "Falco", "GBOOST", "iAssembler", "DeltaProt", "deML", "VALORATE", "HLaffy", "SELAM", "miRDB", "LOSITAN", "DockAFM", "SPARTA", "ConSSeq", "ComiR", "QUDeX-MS", "dbDEMC", "BioBIKE", "SeMPI", "NBDB", "SEQCHIP", "EPGA2", "GenomeTrafac", "TAP", "Harshlight", "RStrucFam", "ProBiS", "BiNGO", "mirEX", "BioGranat-IG", "FLEXc", "SparkSeq", "LocalAli", "COSMIC", "MouseNet", "ppiTrim", "geWorkbench", "Selectome", "TIPR", "PhosphoPICK-SNP", "FERAL", "MRPrimer", "CNV", "ClusterControl", "E-TALEN", "LAST-TRAIN", "NoDe", "Genes2Networks", "cBinderDB", "BioSamples", "TiSGeD", "RIKEN", "ProteoLens", "PBEQ-Solver", "SwissSidechain", "NucAmino", "HARMONY", "DNAVis", "RegPhos", "dbGSH", "NetDecoder", "MRPrimerW", "DockStar", "SASBDB", "sRNAtoolbox", "PhyloPat", "PREDIVAC", "kSNP3.0", "GEN3VA", "NCLscan", "EMDomics", "HitPick", "BigBWA", "PCAN", "TRAP", "DGW", "IslandPath", "wapRNA", "PASPA", "IsoBase", "DFLpred", "slaMEM", "ReportingTools", "Mouse Genome Database MGD-2017", "ProSMoS", "PhenoMiner", "Monster", "WEB-based GEne SeT AnaLysis Toolkit WebGestalt", "JEPEGMIX", "BmncRNAdb", "UniGene", "MetPA", "DNAssist", "XenoSite", "PCP", "MetaboSignal", "EGAN", "BoostGAPFILL", "SCPC", "GENIUS", "blend4php", "SL2", "RiboTagger", "SketchBio", "PocketMatch", "Sealer", "MSOAR", "Bon-EV", "BCL2DB", "ViTa", "NAPP", "UniPathway", "TIGR", "PepBank", "GOblet", "LTR", "PRI-CAT", "DMRforPairs", "CRAFT", "lrgpr", "PlantPhos", "Limbform", "ArrayPipe", "cn.FARMS", "SIRAH", "WikiGenomes", "RINQ", "OCaPPI-Db", "CMAP", "PROMO", "PICCOLO", "Hadoop-BAM", "MPSA", "Qualimap", "CpGProD", "ZiFiT", "VLDP", "MultiFit", "ComK", "@MInter", "cMapper", "OMWare", "MYBS", "GTP", "SCENERY", "puma", "survcomp", "AMIGO2", "ProMoT", "PriorsEditor", "RILogo", "R.E.DD.B.", "KNIME4NGS", "TIF", "BESC", "MEANS", "FunCoup", "MetaVelvet", "LibBP", "'maskBAD'", "PIP-DB", "PyIgClassify", "FLYSNPdb", "HNdb", "antiSMASH", "SBMDb", "FISH-quant", "iRSpot-PseDNC", "CHAOS", "fastER", "ptRNApred", "RSEM", "GWAR", "OASIS", "OntoMaton", "MIRU-VNTRplus", "SINE", "FIEFDom", "coMET", "gEVAL", "StochKit2", "OMPPM", "LIPID", "RGG", "GSAR", "Bio-Object", "DBD2BS", "iPEAP", "BicPAMS", "MeRP", "PANDORA", "OrthologID", "ACG", "INFO-RNA\u2014a", "CUDA", "mirDNMR", "Disperse", "PLB-SAVE", "SubmiRine", "PI2PE", "DCE@urLAB", "CompMap", "Mutascope", "CryGetter", "PRESTO", "TARGeT", "Teolenn", "mkESA", "PCOSKB", "BackCLIP", "NAIL", "MimoDB", "RevEcoR", "QuantProReloaded", "pre-microRNAs", "DBATE", "Omega", "csaw", "USR-VS", "CONAN", "ChemBank", "CellMaps", "QPCR", "MitoProteome", "BepiPred-2.0", "FLASH", "webSDA", "COMBINE", "Dissect", "GeneValidator", "GenomeD3Plot", "SHRiMP2", "DotKnot", "C-SPADE", "CGH-Explorer", "POGO-DB\u2014a", "PyNAST", "PhyloPro", "PICR", "PEDE", "BATVI", "BGFit", "ETE", "GTRD", "PerturbationAnalyzer", "BEsTRF", "CellCODE", "Cildb", "CCRXP", "Consed", "Predict-2nd", "SigFuge", "GlobalMIT", "Ceiba", "CySBML", "PhD-SNPg", "CRISPR-mediated", "PPISearch", "SPADEVizR", "USI", "DendroPy", "XML-based", "BioContext", "TrueSight", "PReFerSim", "CDD", "MutationAligner", "Breakpointer", "MLST", "sc-PDB", "BrainScope", "pBACode", "Phylometrics", "Solubis", "RMBase", "QGRS", "ViralmiR", "Depth", "CSpritz", "Jenti", "Enrichr", "DIALIGN-TX", "baySeq", "FT-COMAR", "RegRNA", "NETTAB", "DSviaDRM", "dbPSP", "HGV&TB", "SPRINT", "FITBAR", "RPdb", "MetaCluster", "VAST+", "XGSA", "SAMPLEX", "eGIFT", "MultiLoc2", "kpLogo", "spongeScan", "EvoTol", "GONOME", "SePreSA", "Blue", "Cloud-Coffee", "PCA", "SnowyOwl", "MetaFMF", "QSSPN", "MAGNET", "RiceNet", "MyProteinNet", "BreaKmer", "SPARQL", "RSEQtools", "SCExV", "Patikaweb", "caOmicsV", "SeWeR", "API", "SDR", "SynView", "OMWSA", "READSCAN", "AliView", "ConoServer", "Brain", "SNOW", "GSearcher", "MULTICOM", "MONALISA", "PhylDiag", "miRPlant", "COGNAC", "RCSB", "UTR", "DNorm", "PathBinder", "Skittle", "GEISHA", "MetaDiff", "ProphNet", "MSMS", "SurvCurv", "BioLinux", "BLAST-based", "A3D", "ComBat", "Bios2mds", "GTRAC", "MycoperonDB", "DBTGR", "EcID.", "SPREAD", "Mireval", "QTLdb", "CharProtDB", "LIGSIFT", "VirusMentha", "FOCAL", "PupaSuite", "eggNOG", "WhopGenome", "dbPSHP", "SVScore", "mimicMe", "EXTRACT", "GeneDB", "ProPepper", "BioMart", "ValidatorDB", "Turtle", "ConceptMetab", "LASER", "aRNApipe", "RAIphy", "ToxoDB", "MitoSeek", "VarB", "CoRegNet", "qvality", "MisPred", "Starr", "MEMO", "DARC", "GLOGS", "OMIM", "Comet-FISH", "AssemblX", "ExonMiner", "InterRNA", "BAM-matcher", "Pythoscape", "BioSAVE", "UniRef", "MetaQUAST", "MBBC", "IQMNMR", "COMET", "e23D", "NAPS", "miRNEST", "GelJ", "MetAmp", "Poly", "MetaboMiner", "Smoldyn", "BioMAJ", "COPRED", "DeepChrome", "BRANE", "EPConDB", "Historian", "ProbFold", "PFGD", "CoopTFD", "GyDB", "SCMBYK", "bdvis", "TRED", "PCI-SS", "MIRIAM", "MetDraw", "PAIR", "PRO", "RAMICS", "MALISAM", "LongISLND", "SNAP", "GenVisR", "FunSimMat", "OGEE", "SuperTarget", "BibGlimpse", "PDB2PQR", "EpIC", "VADAR", "CLAP", "BreakSeek", "BioPig", "CPFP", "mimiRNA", "REBASE", "BioRuby", "NiBLS", "TANGO", "PCPPI", "destiny", "Bosque", "ACT", "PoPMuSiC", "Clonality", "DAVID-WS", "ProTeus", "miR-212/132 expression and functions", "ProGenExpress", "Wessim", "NETASA", "EVEREST", "RADER", "BiDaS", "Balony", "LPmerge", "TogoWS", "IsoCor", "XLinkDB", "BigQ", "MET", "phyloXML", "iTOL", "QA-RecombineIt", "ELM", "LGICdb", "PermutMatrix", "Mocap", "SEM", "ReplicationDomain", "I-Ssp6803I", "PaxtoolsR", "Multi-netclust", "aTRAM", "ChemMapper", "LocSigDB", "BlastGraph", "AH-DB", "Oasis", "Coalescent", "ScanProsite", "GenomeGraphs", "SNP-VISTA", "ProCKSI", "dbPPT", "HvrBase++", "SAMFIRE", "iStable", "miRnalyze", "iRefWeb", "Gclust", "DSK", "LincSNP", "DroughtDB", "pSumo-CD", "IMGT-ONTOLOGY", "TRITON", "MIMAS", "BioPartsBuilder", "FixPred", "TRP", "plasmidSPAdes", "BloodSpot", "RATT", "ChIPXpress", "PolySac3DB", "DHOEM", "WormBase", "RiceVarMap", "MELTING", "MC-GenomeKey", "XIBD", "SMART", "PRSice", "QQ-SNV", "MPSS", "WholeCellKB", "pocketZebra", "HomPPI", "PHACTS", "RuleBender", "ChIP-PED", "Gel2DE", "TURNIP", "CancerMA", "SOLiD", "BioGRID", "meRanTK", "TarFisDock", "PKIS", "Tripal", "ABCGrid", "AmoebaDB", "MetaSel", "CLIP", "ArrayCluster", "MethPat", "SuperToxic", "HangOut", "OKCAM", "AffyPipe", "PepSite", "BOWiki", "PAPi", "GenMAPP", "UniCarbKB", "SPIM", "RiboAbacus", "ERPIN", "TRaCE+", "MolMovDB", "OBO-Edit", "T-BAS", "XLPM", "Snowball", "MeSH", "PEDB", "IMAAAGINE", "iMir", "DMDtoolkit", "EVpedia", "UniNovo", "RetrOryza", "MetATT", "PTGBase", "CompMoby", "MOBI", "GeneStoryTeller", "MutDB", "LoFreq", "GEMSiRV", "PhytAMP", "R-loopDB", "ProteinDBS", "BioAtlas", "ACLAME", "CSI", "D3E", "RENT+", "PROSESS", "PyBamView", "EFICAz2.5", "ABrowse", "CPDB", "TBMap", "trieFinder", "iPhos", "MosaicFinder", "MICA", "GRSDB", "RAxML", "PIAS", "DOMINO", "RankAggreg", "eBURST", "treePL", "Fels-Rand", "BeetleBase", "HotSwap", "Memoir", "FireProt", "PlasmID", "DiffSplice", "ESPRIT-Tree", "TransfactomeDB", "yApoptosis", "LAITOR", "DPTEdb", "BLASTO", "AWTY", "PhylOPDb", "MeFiT", "VirusBanker", "PLEK", "C-It", "R2R", "TimeClust", "Repitools", "FIT", "STON", "GDFuzz3D", "Oncopression", "BirdsEyeView", "E-RNAi", "iMODS", "NGS-SNP", "PvTFDB", "Epock", "BioTextQuest+", "RNAspa", "CPLM", "SigHunt", "PerlPrimer", "BayesPI-BAR", "diXa", "UMD-BRCA1/BRCA2", "OAF", "Newt-omics", "ZeitZeiger", "EpilepsyGene", "Warpgroup", "easyRNASeq", "DESHARKY", "QuACN", "pong", "TreeTime", "Mirinho", "RAID", "ProbeAlign", "ComPPI", "E-Predict", "EvoDesign", "SGoF", "AquaSAXS", "HotPoint", "IPLaminator", "BDSS", "CometCloud", "MIMEAnTo", "EchinoDB", "ALEA", "MTG2", "SVD-phy", "3D-DART", "CoXpress", "DoOPSearch", "Pharos", "GenPlay", "CAPS", "DIANA-miRGen", "OncodriveCLUST", "miRNAminer", "CUSHAW", "METAL", "GO", "GWAtoolbox", "Membranome", "eXamine", "mrsFAST-Ultra", "Bio-jETI", "IEDB-3D", "FNV", "VIS-O-BAC", "NeuroPID", "idTarget", "EGFR", "PASS2", "DRIMust", "MANET", "OTO", "BioXSD", "BDB", "BRONCO", "bammds", "BisoGenet", "RegNetwork", "eSBMTools", "PolyQ", "GeneCOST", "PST", "NeuroTransDB", "PlantMiRNAPred", "PopDrowser", "GeNMR", "BYKdb", "MPBind", "DISULFIND", "PhyloMap", "AAindex", "Onco-Regulon", "PYCHEM", "BiForce", "BIGSdb", "UGT-catalyzed", "NARWHAL", "LOMA", "Agalma", "W.ND-BLAST", "PSAT", "SBEAMS-Microarray", "EuroPhenome", "phRAIDER", "SPEDRE", "BioWarehouse", "GIANT", "nocoRNAc", "MamPol", "TnT", "CSAM", "UgMicroSatdb", "FFT", "ComSin", "PGen", "PDBsum", "CompGO", "QCScreen", "THGS", "GIS", "TSCAN", "COMPARTMENTS", "CypRules", "Snagger", "NPInter", "D2P2", "CUDASW++", "TopHat-Recondition", "lncRNAtor", "MultiChIPmixHMM", "ViQuaS", "Hooke", "BorreliaBase", "WebScipio", "Picante", "PERMOL", "CanvasDB", "IPAD", "Fastphylo", "Hyperscape", "PyElph", "Web-TCGA", "vipR", "PASTA", "Cypiripi", "wFleaBase", "SlowFaster", "CONTRA", "MCScanX-transposed", "CCDB", "VaDE", "PlantTFDB", "PhyloCSF", "PromH", "CytoSPADE", "iCall", "SNAPPI-DB", "LTRharvest", "DeOri", "FILTREST3D", "AgBase", "FBB", "PGAT", "CerebralWeb", "FASIMU", "cPath", "Enrich", "OPM", "HiTSEE", "Coev-web", "BLAD", "StatQuant", "XTalkDB", "MMMDB", "PAST", "PhyleasProg", "OrthoReD", "CAZy", "RegulatorTrail", "MaxSub", "UniHI", "WebSTAR3D", "MetaProx", "iFUSE", "ORFs", "XSuLT", "NeatFreq", "SCANPS", "EXTREME", "GenomeRunner", "FASTSP", "DENdb", "agriGO", "GeneHub-GEPIS", "AStream", "LYRA", "PuMA", "GeneFisher-P", "Skewer", "metaseq", "CTK", "Bi-Force", "InterMine", "ProMAT", "SMITE", "CHROMATRA", "MicroScope", "RefBool", "MolliGen", "Array2BIO", "PeaKDEck", "DiagnoProt", "GMOseek", "CaFE", "Shimmer", "VirusSeq", "ORENZA", "LabelHash", "MetDAT", "MaxBin", "CiVi", "misFinder", "CODEHOP", "TFRank", "CNAmet", "ecceTERA", "RIDOM", "UROBORUS", "TCC", "tRFdb", "BetaSCPWeb", "BLASR", "BDT", "MOAD", "StemChecker", "ToppMiR", "HiTSelect", "LayerCake", "Scater", "PRORP", "HieranoiDB", "SpectralTDF", "Crass", "dbGaP", "XtalPred", "iLoops", "GECKO", "GATK", "Polyphony", "Genepi", "UIMA", "EyeSite", "'CubeX'", "tmVar", "DectICO", "PSI-BLAST-ISS", "T3DB", "SubSeqer", "TEAK", "ChIPseeker", "Genquire", "GenomeCAT", "ORION-VIRCAT", "KiPar", "FermiKit", "SNPdryad", "PoSSuM", "RBPmap", "jMetalCpp", "HHMD", "OxyBS", "miRBase", "QuickMIRSeq", "FAF-Drugs3", "SOAPdenovo-Trans", "ESTclean", "LedPred", "SLiMFinder", "ExoCarta", "EasyGene", "SWISS-PROT", "Enlight", "Genecentric", "ICRPfinder", "Planform", "Meshable", "TopFIND", "PyTMs", "MASS", "CCProf", "PRIDE", "SRM-based", "CLARE", "MMinte", "IC50-to-Ki", "phylogeo", "dictyBase", "XHM", "PrimerDesign-M", "IMPROVER", "DISMISS", "Rabifier2", "SPANNER", "SeqFIRE", "HHMI", "PLecDom", "PerM", "IPO", "GenProtEC", "RNAbor", "YeastNet", "EENdb", "STRING", "GO-Bayes", "DisVis", "CircularLogo", "SharpViSu", "rehh", "BioProject", "OGO", "GOTreePlus", "SoftPanel", "EGassembler", "ERNE", "SETH", "Phyx", "SGAtools", "snp-search", "3D-Coffee", "AluGene", "PockDrug-Server", "CluePedia", "WormExp", "KERIS", "TIARA", "CTX-BLAST", "Hon-yaku", "MaGuS", "SNPhood", "PrepMS", "FOOTER", "Ultrasome", "MS2PIP", "MAGE-TAB", "HIPred", "PuReD-MCL", "UNDR", "iDHS-EL", "dbPTB", "iBeetle-Base", "AbCD", "PROBER", "Assemblytics", "DT40", "ATGme", "ConTra", "Pep-3D-Search", "TCDB", "DGA", "Mosclust", "GOssTo", "pSTIING", "eGOB", "GO-Module", "ngsCAT", "CADLIVE", "atSNP", "TEES", "Bookshelf", "Tax4Fun", "Fruit-Fly", "admixturegraph", "BACA", "PurityEst", "SInBaD", "scphaser", "bioWeb3D", "PMC", "ArkMAP", "IDBA-tran", "GESDB", "CellTree", "FuGE", "ValidPrime", "SNP", "CorMut", "PROSITE", "SV-Bay", "NOA", "ClinQC", "PILGRM", "CRAVAT", "STR", "DiProDB", "BIGNASim", "SeSAW", "PomBase", "BNFinder2", "IPED", "T-RFLP", "MACE", "MamPhEA", "HapFlow", "MIPHENO", "SysPTM", "PentaPlot", "BeCAS", "ABC", "Protegen", "Grinder", "FrameDP", "MINRMS", "MetalPredator", "CNA", "SpEED", "OPTIMIZER", "BPhyOG", "MODBASE", "E3Miner", "AuDis", "ImiRP", "LassoProt", "Canvas", "PSICQUIC", "BioDownloader", "SYSBIONS", "AffyProbeMiner", "QMachine", "IDEAL", "IDGenes", "GHap", "RiboMaker", "CRNPRED", "uMELT", "DISPLAR", "ArthropodaCyc", "HEG-DB", "HAltORF", "PEPSI-Dock", "CARD", "Corra", "SNPdbe", "RECKONER", "FluGenome", "Conpair", "Subpathway-GM", "Panzea", "THOR", "miRmine", "TOUCAN", "INstruct", "AcePrimer", "ALCHEMY", "PPLook", "FRODOCK", "OCDB", "siSPOTR", "iRefR", "HyBrow", "GPCR-ModSim", "Bendix", "SFmap", "MZedDB", "3DRobot", "JACOP", "PredictRegulon", "BioNanoAnalyst", "McGenus", "MAKER2", "CoLoRMap", "ConsensusPathDB", "IDconverter", "snoGPS", "MRHMMs", "CpG", "POXO", "EXACT2", "ICBS", "PBmice", "WebTool", "NET", "DiMmeR", "ProBias", "CPPpred", "PlanMine", "LNCediting", "MARS", "DCDB", "minimac2", "CellML", "FRAGSION", "DRIMM-Synteny", "Bio3d", "genomation", "rcellminer", "PeCoP", "Biopython", "G2D", "PathScan", "DoRiNA", "PredGPI", "QSSPN", "InvFEST", "ProdoNet", "TrSDB", "OncoSNP-SEQ", "BioModels", "curatedOvarianData", "RICORDO", "TFBSTools", "SATPdb", "Phylesystem", "SPADA", "DataPflex", "MEGA2", "EndoNet", "DEPOD", "Pcons5", "LeishDB", "Change-O", "piRNABank", "HUM", "ExAC", "WebMOTIFS", "rHVDM", "TeraStitcher", "AHTPDB", "Pse-in-One", "eF-seek", "ReadXplorer", "NAR", "EMIM", "SNPHarvester", "DeFCoM", "TCGA2BED", "BioSample", "PSSRdb", "Rfam", "LymAnalyzer", "Comb-p", "DMET-Analyzer", "OrgConv", "Trowel", "SCORER", "The super-n-motifs model", "WormBook", "ToxPi", "SECISaln", "SpaCEM3", "pGenTHREADER", "PSI-Coffee", "MINT", "Readjoiner", "AutDB", "DeepGene", "DATF", "Coral", "OVA", "AVPdb", "HHfrag", "PBrowse", "RASER", "iview", "iGNM", "PocketAnnotate", "PyRosetta", "MOGEN", "AUREA", "TargetRNA2", "Hum-mPLoc 3.0", "Reno", "Minfi", "HEXEvent", "SCPRED", "Pipasic", "AIGO", "ProteoSign", "Ciruvis", "Cerebral", "HPD", "CMWeb", "FamAnn", "Cake", "ModLink+", "AnnotateGenomicRegions", "simDEF", "MicroTar", "MoKCa", "EQP", "MultiGeMS", "PhylomeDB", "HLA-typing", "MSQT", "BISQUE", "PDTD", "e-Drug3D", "CUDA-enabled", "BioIE", "ImageJ-MATLAB", "Broadwick", "ChromoViz", "VirHostNet", "phiSITE", "ComiRNet", "motifbreakR", "ITS2", "CGDB", "GSEA-P", "cyvcf2", "OrtholugeDB", "Cordova", "BioM2MetDisease", "DecGPU", "GFam", "miRNAkey", "TfReg", "METAINTER", "HPC-CLUST", "RCADE", "PyBoolNet", "TopDom", "ISACGH", "TMC-SNPdb", "Hobbes", "CGHub", "peptide-MHC", "RBF", "BCSearch", "RCAS", "Gene2Oligo", "HECTOR", "HMMTOP", "MBROLE", "TOPS", "PyETV", "SCOP", "VIDA", "dcGO", "EVOG", "CrossMap", "DDI-CPI", "UCSF", "Bisque", "PyWATER", "SVAMP", "HOCOMOCO", "EROP-Moscow", "3V", "Meta-Analysis", "Lnc2Cancer", "MindTheGap", "ReFOLD", "HistoneDB", "Co-LncRNA", "ScripTree", "Kalium", "MeDIP-HMM", "UET", "DistiLD", "cnvCurator", "LocTree3", "vFitness", "affylmGUI", "FSelector", "RNA-MoIP", "SDRF2GRAPH", "RAPIDR", "SeqAnt", "Pheno2Geno", "M\u00e1s-o-menos", "PRED-TMBB2", "deBWT", "BioBlender", "LCA*", "Seed", "UTRdb", "MEGA-CC", "DSMM", "GPCR-SSFE", "14-3-3-Pred", "MARV", "PanorOmic", "PIA", "AmyLoad", "VCF-kit", "ISMAGS", "TFAM", "CombFunc", "BS-SNPer", "WEBnm@", "CellLineNavigator", "SED-ML web tools", "SSEA", "iPTREE-STAB", "PDBSiteScan", "GeMInA", "PPIXpress", "HEALER", "DrugBank", "SwiftLib", "KEGGanim", "MS-SIG", "MotifLab", "NEArender", "MAGIC", "PoooL", "PSI/TM-Coffee", "rnaSeqMap", "Primerize-2D", "PrDOS", "ThreaDom", "RegPredict", "DiSWOP", "DRAR-CPI", "ReactPRED", "CoP", "Oases", "O-miner", "PAZAR", "APOLLO", "SiteComp", "Scribl", "FunNet", "CyanoBase", "FragViz", "Pybedtools", "openBIS ELN-LIMS", "Companion", "SBMLeditor", "PnpProbs", "OpenTein", "ANISEED", "MathDAMP", "GRASP", "OntologyTraverser", "3DLigandSite", "JANE", "MOTIPS", "MetaGeneAlyse", "Primer3Plus", "Strepto-DB", "stringgaussnet", "BacMet", "mtDNA-Server", "NullHap", "KAAS", "PBWT", "CELLPEDIA", "CircNet", "MobilomeFINDER", "ARKdb", "VIPERdb2", "BOSC", "Cnidaria", "Clusterv", "Epitome", "LigDig", "fpocket", "G-SESAME", "Gencrypt", "REDCAT", "DISMS2", "BioJS", "TEM-1", "JADOPPT", "Metannogen", "DDMGD", "MRUniNovo", "StructMAn", "BatchPrimer3", "mVOC", "EWS-FLI1", "TreeSnatcher", "STORM", "MPrime", "EXPAR", "ReCount", "SRAMP", "MotifNet", "flowCore", "TOBFAC", "CGHMultiArray", "AMPDB", "TreeDyn", "qpcR", "RBP-Var", "NONCODE", "PathWhiz", "compendiumdb", "EMBL-Align", "chipPCR", "BiDiFuse", "miRExpress", "MPHASYS", "ProteDNA", "Fast-GBS", "Calibur", "HAMAP", "CGView", "FoXS", "REPETITA", "GlobPlot", "MotifClick", "NextGenMap", "R3D-BLAST", "ANalysis", "LEON-BIS", "MycoRegNet", "ArchDB", "Savant", "GABI-Kat SimpleSearch", "SAM", "FireDock", "COMBOSA3D", "relax", "OncoSimulR", "BIOCHAM", "PinaColada", "IVA", "NMF-mGPU", "ISPP", "FRABASE", "RNAhybrid", "WaspAtlas", "Prophinder", "ProQM-resample", "Ka-me", "Xwalk", "EcoGene-RefSeq", "ASAP", "mtDB", "DNABarcodes", "WhichGenes", "COME", "BaMBa", "PoreLogo", "CATH", "HexServer", "SimiCon", "FFCA", "ArrayFusion", "CONFAC", "UQlust", "Sungear", "GTOP", "SHARAKU", "HTQC", "MultiPLX", "MIPgen", "Assembly", "MAPPI-DAT", "MinPD", "Formatt", "GlycoPattern", "TrAp", "Mendel", "P2CS", "MACiE", "SEEG", "ProtBuD", "H3M2", "imDEV", "AromaDeg", "R/qtl", "RapMap", "D-HaploDB", "VarMatch", "TFClass", "NRPS", "ENDscript", "KinMap", "HMDB", "SUPERFAMILY", "FAIRDOMHub", "OncomiRDB", "GR-Aligner", "PrediSi", "jSquid", "SEQATOMS", "myGenomeBrowser", "LitInspector", "NALDB", "fastsimcoal", "BriX", "GOChase-II", "PAPA", "UCbase", "TopPIC", "cellXpress", "SherLoc", "AS2TS", "WU-Blast2", "CRISPR-CAS9", "MixNet", "MIsoMine", "NAPROC-13", "Motif-All", "ADReCS", "NCC-AUC", "MitoMiner", "MRE", "Sprites", "3USS", "PVAAS", "TELiS", "CFGP", "RepeatsDB", "RMAP", "GeneProf", "CheShift-2", "BOSS", "QUAST", "SortMeRNA", "CODOC", "TPMS", "PD-D/EXK", "UVPAR", "TCGAbiolinks", "RecA", "TopoICSim", "NaviGO", "GRAMM-X", "PhytoCRISP-Ex", "ArrayExpress", "DriverDB", "Growthcurver", "ProSAS", "PROPER", "QTL", "DOMAC", "T-HOD", "kinDOCK", "AMP-activated", "GASS-WEB", "PDIviz", "AS-ALPS", "STarMir", "metabolicMine", "PIE", "PePr", "MOID", "gSearch", "tigre", "i-cisTarget 2015 update", "STRUCLA", "jpHMM", "HerDing", "fastSCOP", "FUSIM", "TIBLE", "RBO", "HALC", "Willows", "RAPSearch", "RUM", "InPhaDel", "WiggleTools", "NPD", "Pripper", "AnimalTFDB", "ImmQuant", "Dinosolve", "GlycoCD", "RazerS", "TCS", "RAG-3D", "BioJava-ModFinder", "LAMPLINK", "PhenUMA", "iTagPlot", "GenProBiS", "Minimus", "NMPDR", "PigVar", "GenomeView", "Workflow4Metabolomics", "FILTUS", "PDBe", "CHRONOS", "miRFam", "DRTF", "SNPAnalyzer", "Ulysses", "3DGD", "RIPSeeker", "RAD-seq", "AllerTOP", "FUEL-mLoc", "MUBII-TB-DB", "quantiNemo", "DNPTrapper", "BacMap", "F-SNP", "Datamonkey", "BicAT", "PrimerBank", "CAMPways", "PAnalyzer", "Artemis", "Treelink", "Scratchpads", "RPFdb", "SIMMAP", "ccSVM", "ToxDB", "SigniSite", "PHAST", "QCGWAS", "PLPMDB", "TreeGraph", "GPCR", "miREval", "WebCell", "UCHIME", "RGIFE", "RsiteDB", "DISTMIX", "digeR", "rnaQUAST", "LINCS", "IUBMB", "IDP-ASE", "MaXIC-Q Web", "flowPeaks", "PiNGO", "SAQC", "FusionHunter", "SAP", "ModuleAlign", "CATNAP", "PICMI", "Control-FREEC", "EMVC", "ZDOCK", "dbSNO", "TOPS+", "neXtA5", "EpicCapo", "mAPKL", "SchistoDB", "SiGN-SSM", "PHACCS", "microTaboo", "ChemProt-3.0", "ATIDB", "EDAM", "Phyutility", "Lep-MAP", "netClass", "NORINE", "MassTRIX", "ADME", "FastPop", "Peakr", "SCEPTRANS", "MGED", "MetaPSICOV", "ASSIST", "GWASpi", "DTWscore", "SRinversion", "ASCIIGenome", "MG-RAST", "MIDIA", "Galahad", "Phylemon", "RESID", "OligoArray", "SAVoR", "SuperCYP", "Pgltools", "DBM-DB", "seqMINER", "Strainer", "SynSysNet", "PhyloPattern", "SNP2TFBS", "FusionAnalyser", "MetiTree", "Cube-DB", "SigMol", "FRANz", "MrBayes", "cerebroViz", "VIZARD", "INTERFEROME", "Allie", "EDKB", "APPAGATO", "ProKware", "ORFer", "POLYVIEW-MM", "WALTZ-DB", "CollecTF", "Kin-Driver", "SummonChimera", "PRALINE", "WordSpy", "PaleAle", "MIA", "AthaMap", "mapDamage2.0", "MemBuilder", "PathRings", "PREP", "VANTED", "SemMedDB", "DAFGA", "RPBS", "RT-PCR", "Seeksv", "ShoRAH", "RNASeqGUI", "Pycellerator", "3D-SURFER", "comTAR", "BioCreative", "CandidaDB", "HH-MOTiF", "Mulcom", "webPRANK", "Bio", "UCSC", "PSORT-B", "dbPEC", "WorfDB", "Trimmomatic", "MamMiBase", "SUSHI", "HbVar", "agplus", "DiffLogo", "OLS", "CLOE", "MassSorter", "ABRA", "iFold", "KIR", "SoyFN", "ClueGO", "SuRankCo", "IsoMut", "ORF", "EC", "PHYLOViZ", "ENVIRONMENTS", "HapCol", "PharmacoMicrobiomics", "adegenet 1.3-1", "GWAB", "PathWave", "FunShift", "Primer-BLAST", "ProtoNet", "Phy-Mer", "EBI", "*omeSOM", "TFM-Explorer", "tcsBU", "cnvCapSeq", "SNPeffect", "ORGANizer", "jMRUI2XML", "medpie", "CAME", "CS23D", "Antimony", "MetICA", "ALVIS", "LymPHOS", "CytoscapeRPC", "RGmatch", "Genevar", "cgDNA", "LncReg", "ppdb", "ConSole", "RosettaDock", "CoMoDis", "TADtool", "PPI", "PONDEROSA", "BioInfer", "3CDB", "RCMAT", "PORTRAIT", "WiseScaffolder", "eXPatGen", "TACTICS", "MINTbase", "MSV3d", "ProtoBug", "sapFinder", "AMYPdb", "PASSIM", "EMBL-EBI", "nDNA-prot", "VarSifter", "methyLiftover", "AutoSOME", "AS-EAST", "MANTIS", "Cosi2", "ProServer", "G-Bean", "DoMosaics", "IMpRH", "AVIA", "ASSIGN", "H7N9", "TReaDS", "GETPrime", "VerSeDa", "Icarus", "SDAP", "DFVF", "TFBSbank", "GBNet", "Cyclo-lib", "PE-Assembler", "VarioWatch", "MetaPIGA", "SMAL", "molBLOCKS", "BioTextQuest", "PLOTREP", "JEPETTO", "GENOMEPOP", "G4RNA", "ReTrOS", "CREME", "ChromothripsisDB", "PEIMAN", "Starcode", "k-SLAM", "SMARTPOP", "DIME", "wKinMut", "HilbertCurve", "BroadPeak", "CryptoDB", "Orthograph", "GLay", "RS-WebPredictor", "BAC", "PyroHMMvar", "URS", "cl-dash", "RNAcommender", "miRNASNP", "PRODORIC", "VAP", "Biogem", "CAD-score", "KNOTTIN", "VIGOR", "EnrichmentBrowser", "bNAber", "Frnakenstein", "ImmunExplorer", "TMPL", "Edlib", "iPro54-PseKNC", "RMNE", "RSeQC", "PGP", "Chainy", "ASRP", "TreeKO", "SylvX", "NBLDA", "MSEA", "SNPsnap", "CRP", "xHeinz", "ACE", "MOCCASIN", "NextClip", "uORF", "Protael", "LINE-1", "PatMaN", "pfsearchV3", "ORFanFinder", "Mosaic", "UFFizi", "QTrim", "PFAAT", "Simulome", "SuperLigands", "MetExplore", "CG2AA", "BridgeDb", "INO80", "DBCP", "M-CGH", "DBTSS", "neXtProt", "CELO", "CrossHub", "BLOSUM", "MiRNATIP", "TeloTool", "AutoCSA", "QMEAN", "CellClassifier", "RDFScape", "GeneNet", "ABC-SysBio\u2014approximate", "miRmap", "SASD", "KnowLife", "NeuroPep", "POPI", "HMMvar-func", "GBshape", "CoMAGC", "DCGL", "Genomix", "arrayCGHbase", "TargetSearch", "MetaboHunter", "ImmuCo", "EcoCyc", "UniAlign", "CHOYCE", "lDDT", "Evigan", "SA-Mot", "OnTheFly", "BiNoM", "COFACTOR", "G-BLASTN", "SIFTS", "MAESTRO", "qDNAmod", "CAPL", "CpGcluster", "ScaleNet", "SPECtre", "Explicet", "DRYGIN", "SIOMICS", "pROC", "Pfam", "NASP", "ProGlycProt", "DAPfinder", "ChroMoS", "SuccFind", "DOSE", "Pocketome", "LINNAEUS", "mapDamage", "SET", "START", "Helium", "CellMix", "SPA", "REDItools", "BatMis", "GBIS", "COGEST", "ZOOM!", "WegoLoc", "PIVOT", "SpBase", "Alkemio", "CGHpower", "Taverna", "KmerStream", "NONATObase", "BITS2011", "REGULATOR", "PROFcon", "RNAexinv", "SCOUP", "CEpBrowser", "BioPAX", "GO-Elite", "ImpulseDE", "SEGID", "CellML2SBML", "CSB", "MCL-CAw", "PFD", "STOCHSIMGPU", "READ", "D-VASim", "PIBASE", "BRAT", "LUCApedia", "Ligand-Info", "Struct2Net", "TE-Tracker", "RAD", "Calypso", "MetaCycle", "TAP-Tar", "I-PV", "NegGOA", "ONTO-PERL", "NGS++", "PiSQRD", "BetaCavityWeb", "TagCleaner", "JWS", "RNAslider", "EGene", "PEACE", "Fpocket", "JAMSS", "Bind-n-Seq", "GBSX", "ALADYN", "ScreenSifter", "GOParGenPy", "KiT", "JCat", "SPICi", "Sebida", "TAMEE", "SubPatCNV", "WhichCyp", "QualComp", "DIRProt", "R3D", "DNAVaxDB", "PARADIGM", "Sputnik", "PAGER", "MUGEN", "VarySysDB", "SICAGO", "FPC", "PSIPRED", "PDBSite", "PREDA", "openBEB", "KinasePhos", "HMMER", "PhyloPro2.0", "gKaKs", "SNPlice", "SNPmasker", "CellBase", "LASAGNA-Search 2.0", "OLego", "GOexpress", "MoDPepInt", "Romulus", "MAPU", "CEREALAB", "APPEX", "MEPSA", "SwarmDock", "PhosD", "CyanoLyase", "Clotho", "LCR-eXXXplorer", "Mugsy", "eQTL", "ImageBee", "ARYANA", "CAGER", "CHASM", "LNA", "GeneTrail2", "COLT-Cancer", "HTJoinSolver", "miRdentify", "PDBj", "ISCB", "H-DBAS", "gargammel", "MineBlast", "DescFold", "Evola", "Cyber-T web server", "SCell", "PMTED", "PyMix", "PATRIC", "OpenStructure", "ConsPred", "RMD", "MolSurfer", "ChemMine", "SVM-based", "NLStradamus", "sciReptor", "DAS", "FTProd", "Socrates", "3D-GNOME", "MEROPS", "ChiLin", "VANLO", "CRISPRcompar", "BitterDB", "myChEMBL", "PARRoT-", "Riborex", "SimLoRD", "GlycoMapsDB", "essaMEM", "OntoGene", "BANFF", "BioWord", "miRA", "MetaPrint2D", "DDIG-in", "KID", "BitPAl", "UMD-MLH1/MSH2/MSH6 databases", "RNASeqMetaDB", "DPDB", "CATMAID", "SMARTCyp", "Pairagon", "Getis-Ord", "ATAQS", "RJPrimers", "CLARK-S", "SlideSort", "EpiFactors", "CTCF", "SVA-PLS", "GeneRIF", "MINE", "GeneDesign", "GWAS3D", "dbSAP", "RDP-II", "SubtiWiki", "SinQC", "cyNeo4j", "NrichD", "APCluster", "InteractiveROSETTA", "COPASI", "miRror-Suite", "miRNAMap", "PredyFlexy", "e!DAL", "SLPR", "PheNetic", "RNA-TVcurve", "Wildfire", "EMAAS", "MPP", "WSsas", "Hollywood", "FastContact", "RNAtips", "CDvist", "PollenCALC", "PLS", "RBPPred", "REPK", "miR-Synth", "GEPIA", "PathVisio", "methBLAST", "AWclust", "3dSS", "PAR", "MeMo", "DiANNA", "PATHWAY", "M-GCAT", "SECA", "sgTarget", "PETfold", "ASTRAL-II", "ComPhy", "KEGGconverter", "DUDes", "IMG/M", "BiRen", "Kollector", "EMBnet", "ProSAT", "RPF", "tRNAscan-SE On-line", "MHC-II", "REPuter", "PaperClip", "Sincell", "SuperPose", "shinyGEO", "NIH", "APADB", "iEnhancer-2L", "PanOCT", "FARAO", "OligoArrayDb", "Chem2Bio2RDF", "BFC", "SuperTriplets", "CGKB", "MeDuSa", "BIPA", "DARNED", "ancGWAS", "VirOligo", "SoFIA", "ESID", "Biblio-MetReS", "SBMLToolbox", "MAK", "BloodExpress", "TrypanoCyc", "CHOPCHOP", "ARED", "INFEVERS", "DEF", "AutismKB", "SABIO-RK\u2014database", "SIMPROT", "miRTarBase", "Web3DMol", "MIRZA-G", "HCV", "SNPranker", "AGScan", "POPISK", "BioDrugScreen", "Hydra", "ParTIES", "SPNConverter", "GenDecoder", "HapMuC", "miRGate", "LASAGNA", "MyBioNet", "MiRFinder", "HDOCK", "CROC", "3PFDB+", "dbSUPER", "BASE", "GelScape", "densityCut", "rSeqNP", "FastML", "PELE", "TSTMP", "TABASCO", "TAPDANCE", "MagicViewer", "ChiBE", "ApiDB", "CLICK", "RNA-seq mixology", "ChEA", "WebGIVI", "SimRNAweb", "OpenChrom", "3D-Garden", "Tavaxy", "Xtalk", "MarVis", "ELASPIC", "PILER-CR", "NanoStringNorm", "WAXSiS", "GPS-Prot", "PicXAA-R", "SMETHILLIUM", "VarSim", "ASH", "metaTIGER", "BLSSpeller", "EvolView", "MolProbity", "Memdock", "RAPIDO", "PEP-FOLD3", "CyDAS", "geneCBR", "PAML", "RaCe", "DISOPRED3", "PSEUDOMARKER", "Patrocles", "Ontodog", "Function2Gene", "VectorBase", "ChloroMitoSSRDB", "ALFALFA", "DiseaseMeth", "GiardiaDB", "BioLingua", "DAVID", "aRrayLasso", "ArchTEx", "QUBIC", "MIPS", "birgHPC", "SimpleSynteny", "AtNoPDB", "SNiPlay3", "GOA", "SWISS-MODEL", "zfishbook", "Transposome", "VKCDB", "YEASTRACT-DISCOVERER", "FindPeaks", "SetRank", "PathText", "NOREVA", "BindingDB", "MetalDetector", "NutriChem", "Recco", "freeIbis", "InterEvScore", "QuickTree", "MUMMALS", "NOBLE", "EFICAz2", "DoGSD", "ConservedPrimers", "subSeq", "LedPred", "ProtPOS", "ProMoST", "NBRP", "MetaCyc", "HiTEC", "CasOT", "Meander", "SurvNet", "ParsEval", "AAARF", "FABIA", "BioSharing", "MOSAIC", "probeBase", "TPMD", "WGLRH", "PDB-REPRDB", "AppaDB", "LRpath", "Arcadia", "DFBAlab", "DFLAT", "RIsearch2", "Pan-Cancer", "Seq2Logo", "Pepper", "FAVR", "PTMTreeSearch", "NRE", "PROTEOFORMER", "MAPSP", "GenomeCons", "ExpTreeDB", "PaGeFinder", "REIDS", "dbDEPC", "SBML", "SOBA", "GenePainter", "SUNPLIN", "BioQRator", "PlnTFDB", "NeuroPedia", "pez", "WebGestalt", "LDSS-P", "LiverCancerMarkerRIF", "Trinculo", "JSpeciesWS", "JenAge", "Shiny-phyloseq", "PSIKO2", "Osiris", "Blast2GO", "NAGNAG", "EpiDBase", "Morpheus", "AlignGraph", "B-SOLANA", "cMonkey2", "AllelicImbalance", "Flower", "SurvMicro", "MERAV", "CSA", "viRome", "LoopIng", "Pclust", "Orthoscape", "Bayexer", "Mollack", "Gramene", "KDDN", "PortEco", "Combenefit", "SNPGenie", "Exposome-Explorer", "RASS", "RNAiFold", "AbMiner", "EPEPT", "PowerMarker", "SCRIPDB", "SIMToolbox", "BRAGI", "EXCAVATOR2", "GenBank", "Seq2pathway", "SubtiList", "HemeBIND", "RNAPattMatch", "KNIME-CDK", "EDGAR", "gcExplorer", "Satellog", "DIANA-TarBase", "iHOP", "SuperPred", "M3-S", "ValidNESs", "Aber-OWL", "STRUM", "BioCyc", "miRspring", "ViroBLAST", "SPINAL", "ChemTreeMap", "PDBest", "EricScript", "MTML-msBayes", "BUSCO", "BubbleTree", "PEP-SiteFinder", "CAMERA", "TOPPR", "SNPFile", "RNAMotifScan", "FastProject", "BEADS", "BRENDA", "QDREC", "GoMiner", "ALFRED", "ProbeMaker", "SZGR", "ChiNet", "JED", "Rclick", "PLAST-ncRNA", "RuleGO", "TeloPIN", "BioJava", "HMM-FRAME", "tRNADB-CE", "DGVa", "SomInaClust", "NetworkAnalyst", "BALLView", "FFAS", "CloudMan", "gff2aplot", "SuMo", "GESS", "MODEVO", "SynBioSS", "PatA", "MAGNOLIA", "SeqTrim", "BaCelLo", "PRUNE", "ProtDCal", "PlantsP", "MSstats", "miEAA", "GlyTouCan", "Assocplots", "BEDOPS", "PSORTb", "GeneViTo", "Expitope", "Full-parasites", "DyNet", "JASSA", "SplicingCompass", "qsubsec", "Pysim-sv", "yMap", "HotSprint", "iDBPs", "ESTminer", "HGVA", "genipe", "dbDSM", "Phylocom", "CellH5", "ATTED-II", "DOOSS", "TFInfer", "DEEP", "CDK-Taverna", "InDeVal", "ParCrys", "Cleaver", "GenRGenS", "Payao", "Recycler", "UGT-mediated", "Sanjeevini", "DelPhi", "KeBABS", "SymbioGenomesDB", "GEO", "CPSP-web-tools", "ViralORFeome", "DecoyFinder", "BioAfrica", "MODEM", "GB3.0", "BAMQL", "IUPHAR-DB", "PeptX", "PTESFinder", "cellHTS2", "IMP", "FANTOM", "CFTR", "RegaDB", "DSRC", "pyGCluster", "Paircoil2", "MOIRAI", "TopNet", "ProtocolNavigator", "DISSECT", "ADNI", "NetCoffee", "SCMBYK", "INTAA", "CDAO-Store", "hsphase", "CoryneRegNet", "CRM", "meta-PPISP", "FFPopSim", "DMEAS", "orthAgogue", "SInC", "miRNAmeConverter", "CCTOP", "CPHmodels-3.0\u2014remote", "TarPmiR", "L-GRAAL", "Sphinx", "RIBFIND", "NASSAM", "FATCAT", "BMC", "SFREEMAP", "BDVal", "PCM-SABRE", "DisBind", "DADP", "DIAL", "SRAdb", "DEDB", "MOLGENIS", "PUDGE", "CutDB", "jMRUI", "BESST", "histoneHMM", "MTiOpenScreen", "Light-RCV", "NOXclass", "Circleator", "RNIE", "iMapper", "deepTools2", "PROVEAN", "OREMPRO", "FBA-SimVis", "Hubba", "PrePPI", "HADDOCK", "PreDREM", "NSDNA", "Metingear", "MetaboLights", "InMoDe", "PLIP", "SOMP", "PCDB", "BLESS", "Conveyor", "MetalionRNA", "CGH", "FRAGS", "GATHER", "LIFEdb", "BAli-Phy", "CASP11", "BioBrick", "SNV", "BAGET", "PPI4DOCK", "ORCA", "CAMPAIGN", "MPID-T2", "tEFMA", "proTRAC", "NNDB", "PANTHER", "EzArray", "PathVisioRPC", "EnzyMiner", "GigaDB", "OMICtools", "iATC-mISF", "RiPPMiner", "MetTailor", "GESPA", "PathVisio-Faceted Search", "Inparanoid", "SimRNAweb", "CNV-seq", "Redundans", "iMod", "emPAI", "Skyline", "GOplot", "MATLIGN", "ARACNe-AP", "MASISH", "PrionW", "psRNATarget", "Phospho3D", "inGAP-sv", "Simcluster", "MolBioLib", "QVZ", "GeneMANIA", "eTUMOUR", "OMPdb", "KAPPA", "LARVA", "ADMIT", "EM-SURFER", "AltAnalyze", "Bio-Community", "LOCATE", "PMut", "MALDIquant", "DroSpeGe", "OncoSearch", "GC-Profile", "HPID", "AFRESh", "TBestDB", "MVIAeval", "denovo-db", "MSMExplorer", "ALTER", "G-compass", "BioExtract", "SOAPfusion", "OncomiRdbB", "MetaDisorder", "SPARCS", "FTSite", "HypoxiaDB", "FGDB", "SAMstrt", "YADA", "Flexible-meccano", "Haploview", "GenomeHubs", "DECOD", "SPINGO", "BAPS", "ISA", "SEED", "bio-samtools 2", "NMSim", "AllergenFP", "PARS", "D-Light on promoters", "Zerone", "NETGEM", "KIRMES", "CMView", "ellipsoidFN", "MetExtract", "MetaShot", "MITOMAP", "ZPS", "TLM-Tracker", "tYNA", "novPTMenzy", "SNPman", "FLOSS", "OCTOPUS", "PANDAseq", "PIANA", "pathDIP", "UniFrac", "GabiPD", "2P2Idb", "VaccImm", "PNRD", "FastMEDUSA", "MicrobesOnline", "ANGES", "TFmiR", "rHAT", "Heatmapper", "YOC", "MixMir", "FERN", "LocateP", "IBS", "COFECO", "PHARMACOLOGY", "TIPdb-3D", "ReLA", "cuticleDB", "PedMerge", "ProFunc", "SIMBA", "PLATINUM", "QueryOR", "CopyCat", "HiLive", "FSRD", "HuntMi", "PEAR", "VASP", "Protter", "Phamerator", "TropGeneDB", "AnnTools", "J-Express", "iPBA", "euHCVdb", "interPopula", "EPSILON-CP", "MultiSeq", "ContEst", "INCA", "QUMA", "ClustVis", "MicroPIE", "PEAKS", "PAIDB", "CHOmine", "WEGO", "DBTBS", "i-GSEA4GWAS", "UK10K", "iMembrane", "SAMBLASTER", "TDR", "GSEA-SNP", "MetaRef", "SANS", "VSDocker", "JRC GMO-Amplicons", "ARGON", "BmTEdb", "AGGRESCAN", "PRince", "GMcloser", "PCTFPeval", "Pepitope", "ReformAlign", "iSARST", "Mutadelic", "ALE", "GEMINI", "AnyExpress", "PON-mt-tRNA", "CatchAll", "Assemble", "DIANA-mirExTra", "PheMaDB", "Pandit", "FunFrame", "PHASTER", "ContextD", "modENCODE", "StructureFold", "GEM2Net", "NET-GE", "SIBIS", "I-TASSER-MR", "PiSite", "TFinDit", "BayGO", "ENViz", "ATria", "ChromoHub", "DBSI", "AlgoRun", "VariantAnnotation", "MMsINC", "nbCNV", "BEAN", "PESCADOR", "iPARTS2", "JET2", "Nucleos", "BadiRate", "RDML", "LocusExplorer", "DASP3", "MetaMapR", "SILVER", "PROMPT", "GraphCrunch", "GrainGenes", "VOE", "MSAProbs", "NPDock", "MMFPh", "DNAlive", "5\u2032-GTA/C", "BLAST", "IDEOM", "SITEHOUND-web", "YEASTRACT", "5SRNAdb", "dendextend", "bTSSfinder", "TaxMan", "CAGEd-oPOSSUM", "KnotInFrame", "HORI", "InCroMAP", "FastMG", "OmicBrowse", "Adepth", "StrAuto", "ViRBase", "COMPASS", "JiffyNet", "GenePublisher", "VAMPS", "Meta-Storms", "CRISPR", "sfinx", "STRAND", "ADPriboDB", "CCLasso", "CIS", "NAViGaTOR", "CNV-WebStore", "diffHic", "GView", "TSSer", "AIDA", "Tailor", "QuickGO", "Sarment", "UL8", "ASAFE", "ksRepo", "RefSelect", "SoyNet", "PeptideAtlas", "UGENE", "partDSA", "GeNemo", "ROTAS", "PCFamily", "GOSSIP", "AraC-XylS database", "MetCap", "RetroSeq", "Roar", "Cell-Dock", "DASMI", "PennSeq", "KiDoQ", "BioHealthBase", "OnlineCall", "SORTA", "DupTree", "GUUGle", "BiSearch", "VariScan", "footprintDB", "PhagesDB", "KA-SB", "POOL", "TOPSAN", "Oculus", "BRB-ArrayTools", "UMLS", "Path2PPI", "PostMod", "CLImAT", "SYMBIOmatics", "SADI", "TENET", "FUNNEL-GSEA", "YTRP", "AGeNNT", "CASSIS", "SALT", "MaGe", "MoRFs", "IRES-mediated", "cgmisc", "JPred4", "ChemBioServer", "Brunn", "GREGOR", "muBLASTP", "MIEC-SVM", "TSEMA", "CollapsABEL", "FREGAT", "DeuteRater", "dbPTM", "yMGV", "mTCTScan", "YASARA", "ChemEx", "MEGA-V", "HExpoChem", "PhiSiGns", "RISCI", "BioBlend", "NetDS", "GROMACS", "kpath", "CTLPScanner", "DSSTox", "SBSI", "SecureMA", "jPREdictor", "GenoWAP", "TOPDOM", "SYSTERS", "Fitchi", "MEvoLib", "MPDA", "TTP", "Rainbow", "VDJML", "MEMOSys", "GOstats", "human-gpDB", "BioVLAB-MMIA-NGS", "variancePartition", "Sim4cc", "XML", "JigCell", "MSigDB", "TIGERi", "ROS-DET", "ProbeSelect", "CRISPR-Cas", "SpotCaliper", "PromoterCAD", "PatternQuery", "PCE", "FusionMap", "Hybrid-Lambda", "BLAST+", "LocalMove", "NetPathMiner", "TTSMI", "ANAIS", "SIGNOR", "FindPath", "GWAF", "Pathview", "NCMine", "RIPCAL", "SEQualyzer", "STRBase", "Jane", "Compo", "StreptomeDB", "PmiRExAt", "miRNet", "IDSS", "BIS2Analyzer", "PBOOST", "LibSBML", "FlyNet", "COXPRESdb", "REDfly", "oPOSSUM", "KIDFamMap", "SNPs", "NeXO", "CMA", "PhytoPath", "A-Cell", "ProfDist", "RNALOSS", "SNPedia", "VariantBam", "Meta-Alignment", "Simulate_PCR", "R-chie", "Tasuke", "PDA", "MODexplorer", "GCView", "lodGWAS", "IPAVS", "BioBayes", "RegulonDB", "MatGAT", "MetMSLine", "KEA", "Design implementation and practice of JBEI-ICE", "PhyloPars", "CellFinder", "SIST", "HESAS", "HiXCorr", "SpolPred", "Ribostral", "lncRNome", "SwiSpot", "BioMoby-based", "FUNYBASE", "Evoker", "Oqtans", "CRT", "miROrtho", "BioLexicon", "VCF2Networks", "SurpriseMe", "Path", "r2cat", "MutaGeneSys", "ssSNPer", "PSE-HMM", "EGAD", "3Drefine", "EnsCat", "PathwAX", "MiCoViTo", "SetCD", "Mango", "LOCUST", "MAIA", "Sigma", "PROTEIOS", "OligoWalk", "Quad-SNP", "ANM", "FINDbase", "Eureka-DMA", "VAMP", "msBayes", "GWIDD", "UniSave", "PhytoClust", "EMBL", "SoyKB", "BasyLiCA", "K-Fold", "DTA", "yStreX", "VisANT", "WWW", "SBGN", "GGRNA", "RNAsoft", "VISIBIOweb", "DNABINDPROT", "Indelign", "CC+", "IRESite", "MuCor", "PMC-SVM", "Pscan", "sdef", "HAMMER", "NetSlim", "PepSplice", "LAMSA", "Musket", "Cardinal", "BEST", "PGS", "GenoQuery", "CPAT", "MatInspector", "BioAcoustica", "QuASAR", "T-lex2", "MPRAP", "M-ORBIS", "ngsTools", "ECOD", "RhesusBase", "K-OPLS package", "Narcisse", "MiCroKit", "t2prhd", "GOBASE", "groHMM", "T-RECs", "GeneCodis3", "RVTESTS", "FoCuS-point", "BIOCAT", "DNATCO", "SPRITE", "HAlign", "ECR", "RADIS", "RTPrimerDB", "MLPAinter", "MRP", "TipMT", "PHOSIDA", "Screensaver", "NetAcet", "tDRmapper", "BioJazz", "IRIS", "NetworkPrioritizer", "AGenDA", "ProteomeXchange", "iPfam", "RSAT", "MathSBML", "GlyProt", "GeneBrowser", "SLICE", "FlexPepDock", "CNstream", "DASher", "FungiFun2", "plethy", "Gendoo", "svmPRAT", "RLIMS-P", "RaptorX-Property", "Doelan", "GenAlEx", "IslandViewer", "KAT", "CUDAMPF", "EST", "andi", "LenVarDB", "EPSILON", "ITD", "mGOASVM", "AIMIE", "affy2sv", "SIFT", "Cassis", "COUSCOus", "eFIP", "wuHMM", "VIRmiRNA", "IsoFinder", "EDDY", "PURE", "MeCorS", "StralSV", "3D-footprint", "SDOP-DB", "AlmostSignificant", "KNODWAT", "UKPMC", "sMOL", "Patch-Surfer2.0", "PriFi", "WeGET", "AKE", "ProtVirDB", "BBCAnalyzer", "HyPhy", "TIGAR", "BSDD", "POLYVIEW", "PEDSTATS", "BiQ", "SAG-QC", "Parasail", "miRseqViewer", "SuperHapten", "Lnetwork", "RDP3", "SFESA", "MicrobeDB", "microPIR2", "SpartaABC", "fCCAC", "SiTaR", "BSDB", "PLAAC", "MITIE", "iNuc-PseKNC", "HighSSR", "2dSpAn", "CESAR", "ProQ2", "ESCAPE", "eSLDB", "DNAproDB", "ISVASE", "MedCrawler", "PROTEUS2", "ParseCNV", "linkedISA", "ntHash", "StarScan", "Vizardous", "GALANT", "Allegro", "ORMAN", "CD-Search", "TCGA2STAT", "CGAT", "ValiDichro", "STAMP", "PHEMTO", "SeqAssist", "C-It", "AISAIC", "SinEx", "GOBICS", "CORNA", "GenomeRing", "PlanktoVision", "RNAG", "VCFtools", "SNP-o-matic", "Ferret", "HyLiTE", "ToPASeq", "EXPANDER", "UniCarb-DB", "RegPrecise", "BinDNase", "DDGni", "GOChase", "BIC-seq2", "cisRED", "BioPhysConnectoR", "SegMine", "COCOS", "INGA", "Pathbase", "SuperNatural", "YASS", "BeEP", "DIP", "annot8r", "Metriculator", "mcaGUI", "MPI-LIT", "SPANG", "BLANNOTATOR", "HemaExplorer", "shortran", "TMM@", "PSICOV", "LocNES", "SarConfoCal", "NXSensor", "dbWGFP", "U12DB", "\"\"\"TOF2H\"\"\"", "SuperMimic", "MachiBase", "MGIS", "EMSAR", "Onto-Tools", "MPSQ", "PatMatch", "SiteHound", "ERC", "Biana", "funtooNorm", "RibEx", "PiHelper", "DIANA-miRPath", "GTB", "Dendroscope", "PROMALS", "SABBAC", "MouseBook", "ModeRNA", "SATuRN", "BMix", "SETTER", "PathVar", "Martini", "Arpeggio", "AfterQC", "MEGA-MD", "Primique", "DASS-GUI", "GMD", "NGS", "UniProtKB/Swiss-Prot", "TMA", "MPact", "BCRgt", "PK/DB", "iCR", "dbSNP", "GPCR-specific", "ProteoWizard", "tagtog", "SMMRNA", "PepComposer", "pcaReduce", "AutoGRAPH", "CABS-flex", "KOMODO", "Epstein-Barr", "PathVisio-Validator", "KOSMOS", "ABAEnrichment", "PON-Sol", "LAMARC", "DiNAMIC", "DBatVir", "KEGG", "plantiSMASH", "AraNet", "UUCD", "ECO", "Array-A-Lizer", "POLYPHEMUS", "Hi-Corrector", "GBSA", "PathGen", "ORegAnno", "LocDB", "rrnDB", "TopiaryExplorer", "sgnesR", "CTree", "Absynte", "Multi-Q", "CottonGen", "BinAligner", "DEOGEN2", "Geoseq", "DKFZ", "SIFTER", "CentiLib", "CSTEA", "ArrayMining", "TIGRFAMs", "PySCeS", "3did", "FaQCs", "STSE", "FACT", "PSCDB", "seeQTL", "TSSAR", "MetRxn", "SurreyFBA", "CytoSQL", "multiSNV", "HMCan", "BicSPAM", "MEDock", "MegaMotifBase", "SIRW", "pProRep", "PINTS", "CoFold", "PeroxisomeDB", "Pre_GI", "R-SAP", "Nonpareil", "SMotif", "SpA", "GEneSTATION", "enoLOGOS", "RNA-Redesign", "TMalphaDB", "NetExplore", "IPEP", "PheWAS", "GermSAGE", "DoOP", "WAR", "EXIMS", "SteinerNet", "Groups", "PhosPhAt", "PaLS", "QMSim", "TESS", "CONTRAfold", "MeMotif", "TREE-PUZZLE", "SPARQL-enabled", "H-Invitational", "MetaCRAM", "FoldMiner", "HMCan-diff", "MovieMaker", "G-SQZ", "AnnotQTL", "PlantTFcat", "P3BSseq", "seeMotif", "MetaRanker", "Nozzle", "UbiProt", "InCoB", "DoBo", "SEPIa", "QoRTs", "TT2NE", "AutoDock", "DOCKSCORE", "APoc", "Transcriptional regulation ofElf-1", "NGS-Trex", "MUSCLE", "Gene", "JColorGrid", "LabKey", "mutLBSgeneDB", "sCNAphase", "VIRsiRNAdb", "UNRES", "QTREDS", "CHEXVIS", "AnnotationSketch", "SMD", "RefSeq", "Diffany", "CRISPR-DO", "kmacs", "BusyBee", "ImageJ", "Meta-eQTL", "CORG", "MZmine", "Sigma-2", "ClaRNA", "HitWalker2", "proTF", "Maligner", "iBioSim", "ECOH", "GLIDA", "NetworKIN", "ParDRe", "CAPS-DB", "SIRENE", "VARNA", "HMM", "MtbRegList", "easyDAS", "MMR", "Asterias", "D3M", "Ricebase", "DFP", "Whiteboard", "RECON", "AMEN", "GeneFriends", "SCIFIO", "R-Gada", "HLA", "easyFRAP", "KalignP", "3DMolNavi", "MEPD", "ASG", "PANNZER", "PMRD", "dRiskKB", "e-LEA3D", "SelTarbase", "NHPRTR", "RAxML-Light", "Bpipe", "PepServe", "SAMNetWeb", "pssRNAMiner", "NetiNeti", "BiNChE", "CyClus3D", "MLML", "VASCo", "SyntTax", "ERD", "PAS", "SigMod", "EBSeq", "NetwoRx", "PARALIGN", "spliceR", "HMGB1", "W-IQ-TREE", "PupDB", "HPMCD", "SplitMEM", "HSPIR", "C2H2", "Shape-IT", "InterMap3D", "EmpPrior", "ITTACA", "ICC", "WGE", "CHiCP", "Wurst", "ANNIE", "TraDIS", "MesoRD", "Ct3d", "tcR", "ProtVista", "Template-based C8-SCORPION", "ChopSticks", "NCI-60", "RNAmutants", "WaveCNV", "SLiMSearch", "SMPDB", "MetaProb", "RNASequel", "iPPI-DB", "systemPipeR", "ScreenBEAM", "OCSANA", "GOmotif", "ShrinkBayes", "TraceEdit", "AspGD", "XperimentR", "PSAR-Align", "NURD", "GPU-Meta-Storms", "PBIT", "G-language Genome Analysis Environment", "FTMAP", "CoSSMos", "CD-HIT", "InDel", "MAFFT", "Noncoder", "DARA", "Ligandbook", "TRFolder-W", "MotiveValidator", "CDRUG", "ProClaT", "DrugCentral", "PROMISCUOUS", "IBM", "Faspad", "BALL-SNPgp\u2014from", "PTID", "CouchApp", "BSeQC", "Skylign", "DECIPHER", "iGEMS", "PhyloJIVE", "BIPS", "SAPIN", "Bridges", "BioNet", "ESpritz", "GMO", "iCTNet", "RNAdualPF", "TcoF-DB", "NATsDB", "BioWareDB", "Meta-All", "Ori-Finder", "ChiTaRS", "HubMed", "mirConnX", "VirusMentha", "DNAApp", "Ultranet", "Seqcrawler", "DiscML", "CoLoMoTo", "MergeAlign", "PatchDock", "HDTD", "TarBase", "GS-align", "ChIP-exo", "NRG-CING", "ISCB-Asia", "Segmentum", "2DDB", "PredUs", "REDEMPTION", "Progenetix", "BARD", "GenomeVx", "SplicePlot", "Ontobee", "BRANCH", "NGSCheckMate", "SVA", "MorphoLibJ", "MEDLINE", "TRES", "CompareSVM", "CloudBurst", "DINIES", "NetBenchmark", "VIPR", "r3Cseq", "PanGP", "lpNet", "PottersWheel", "Broad-Enrich", "COSINE", "MnTEdb", "MuteinDB", "MethPed", "PChopper", "FunSpec", "MoNetFamily", "GMOD", "PIPs", "MCMC", "CpG_MI", "SDM-Assist", "miR-PREFeR", "Nebulon", "TCGA", "Alpha-CENTAURI", "SPINE", "Pathomx", "Annocript", "TransportDB", "SCALCE", "SiteSeer", "NightShift", "MSProGene", "ReQON", "CPSP-tools", "WALT", "Metab", "aCSM", "Pro54DB", "DSigDB", "EuPathDB", "GenColors-based", "KnotProt", "SLiMScape", "SOAP3", "QARIP", "al3c", "Var2GO", "AAgAtlas", "tHapMix", "AbsCN-seq", "isomiR-SEA", "FACIL", "SBML-SAT", "RNASurface", "ASDB", "FEELnc", "BiSciCol", "TFcheckpoint", "RAREMETAL", "gtrellis", "PathoPlant", "SeqDoC", "CancerPPD", "BioASF", "FlexServ", "YAHA", "EMPReSS", "CIIPro", "GOR", "PROMALS3D", "optiCall", "STITCH", "WikiPathways", "W-ChIPMotifs", "AQUA", "NHGRI-EBI", "MREdictor", "FoldX", "SpliceNet", "TomatEST", "Hapl-o-Mat", "INTEGRATE-neo", "Cscan", "J-Circos", "Glimma", "HOPPSIGEN", "CoEPrA", "CePa", "DPRP", "ChiPPI", "PedGenie", "MToolBox", "Ub-ISAP", "BSPAT", "YOGY", "NET-SYNTHESIS", "KAnalyze", "PathVisio-MIM", "MTide", "ShiBASE", "I-COMS", "DBTMEE", "TreeToReads", "CancerResource", "CCancer", "MetaTM", "ClonoCalc", "Mechismo", "NimbleGen", "CREDO", "ExportAid", "HotRegion", "PiPreD", "GGBN", "PHISTO", "KGBassembler", "EMAGE", "SAbPred", "DNACLUST", "CAGE", "DG-CST", "SHAP", "ColonyTrak", "PSAR", "SIB", "ParticleStats", "PLIC", "iHAP", "TiPs", "PROCAIN", "PIECE", "SuperSite", "BlockClust", "ADEPT", "ArrayWiki", "CoPAP", "ChainRank", "IntFOLD", "PALSSE", "GeneScissors", "ProLoc-GO", "NSSRF", "HTS-Corrector", "PlantDHS", "PredHS", "FastEpistasis", "Gepoclu", "OWL", "Metab2MeSH", "MethSMRT", "VIOLIN", "EnhancerAtlas", "InteMAP", "BIOZON", "BIRI", "PathoScope", "BEDTools", "CSI-OMIM", "LMPID", "pKNOT", "PRAP", "chopBAI", "PILER", "PHOCOS", "LegumeIP", "quantGenius", "Halvade", "OpenMS", "BIMA", "ExPASy", "primetv", "LICRE", "NEBcutter", "microRNA-target", "SegCorr", "Voronoia", "TALEN", "DisCons", "VEXOR", "geneCommittee", "PhyLAT", "MetaMLST", "RPI-Pred", "PubChem", "Threshold-seq", "GAGE-B", "SLiMDisc", "FAST", "hiHMM", "PINT", "Seqcrawler", "YM500", "MapMi", "DeNovo", "KBDOCK", "STCDB", "EpiChIP", "Pvclust", "RRBSMAP", "SGFSC", "switchde", "SNVHMM", "SAM-T08", "IEDB", "FMAP", "MPIDB", "CandiSNPer", "EpiGPU", "MonarchBase", "RVboost", "SNP-SNP", "Intervene", "BactPepDB", "Vecuum", "ARTS", "clusterMaker", "DySC", "VarDict", "JunctionViewer", "BatchQC", "pRESTO", "OReFiL", "NoD", "PSI-MI", "NetLand", "SNPs3D", "InSatDb", "DCJ", "COHCAP", "cnvOffSeq", "ESPRIT", "GenNon-h", "Pathema", "HopBase", "EnsembleGASVR", "D-Tailor", "MethylomeDB", "Gene-Ontology", "SUMO", "STAR", "DigSee", "SOURCE", "Circoletto", "CARNAC", "FluxFix", "HTPheno", "CeCaFDB", "ANGSD", "SNiPA", "ParaPep", "PCR", "E-MEM", "OrfM", "BDTcomparator", "NRGC", "IsotopicLabelling", "OBO-Edit\u2014an", "PeakXus", "EpimiR", "msa", "VIBE", "FX", "microRNA-gene", "repgenHMM", "MetaQuery", "MPD", "BioFabric", "FELLS", "I-TASSER server", "SBR-Blood", "Gepard", "HGVS", "CycADS", "ChEBI", "PROCOGNATE", "CRF-based", "PlasMapper", "pHMM-tree", "DoriC", "ClusCo", "CMASA", "SILVA", "MESHI", "DupliPHY-Web", "ESTHER", "ProteoModlR", "AtRTPrimer", "rCUR", "TOGGLE", "TED", "ATGC", "miR2Disease", "HyDRA", "FadE", "CFM-ID", "Mass-Up", "lncRNASNP", "PathBLAST", "Chimira", "cDNA-to-genome", "SecReT4", "WebLab", "DNAReplication", "CPred", "SMeagol", "MEDELLER", "iLAP", "ChAsE", "MEGAHIT", "APOPTO-CELL\u2014a", "SNP2CAPS", "ABNER", "SH3", "ILP-based", "RARGE", "Treevolution", "Olorin", "oneChannelGUI", "Scotty", "IMPaLA", "AgdbNet", "HorA", "GWIPS-viz", "Bhageerath-H", "SeqGSEA", "POWER", "PUG-SOAP and PUG-REST", "GlycoRDF", "eSTGt", "PRINTS", "ChIP-Array 2", "DANCER", "Reactome", "Eigen-R2for", "ICM", "COMBREX-DB", "SE", "MethHC", "CVTree", "lncRNA-disease", "SIMAP", "PIUS", "ms-data-core-api", "AutoSite", "FUNC", "miRGen", "KCMBT", "Complexity", "CytoSpectre", "BSRD", "APPRIS", "GeneXplorer", "cneViewer", "OryGenesDB", "Reprever", "OligoGenome", "MICheck", "TimeTree2", "AtPID", "Phytome", "SCARPA", "RepeatExplorer", "MitoCarta2.0", "PocketQuery", "LANL", "BioGPS", "HTT-DB", "BNArray", "QCEWAS", "PANDIT", "EuG\u00e8ne-maize", "HASP", "AlignWise", "SynDB", "MicrobiomeAnalyst", "Ebbie", "RAMPART", "BiasViz", "Rtools", "SCMPSP", "ViaComplex", "PhyloDraw", "MyMiner", "PINTA", "PoPMuSiC-2.0", "ArchPRED", "Pollux", "PlasmoGEM", "GSVA", "Vienna-PTM web server", "NeatMap", "CySBGN", "SwissTargetPrediction", "Epitopia", "OpenSegSPIM", "ExTopoDB", "iCORN", "CRISPR-ERA", "TopoGSA", "BARCdb", "EvoDB", "ClassifyR", "Epidaurus", "GATB", "GERV", "DNAshape", "AMIGene", "Cobweb", "WDDD", "FishingCNV", "GRASPx", "CARMAweb", "MetaGene", "Hi-Jack", "Meta-IDBA", "Ceres", "RosettaDesign", "TISdb", "CCSI", "LIGAND", "SM2miR", "GPSy", "BM-BC", "MosaicHunter", "CytoHiC", "eIF5A", "CEBS", "PRISE2", "HELIQUEST", "PUGSVM", "SwissBioisostere", "BioNLP", "ReplacementMatrix", "ADAN", "mirTools", "SitEx", "BioC", "wwLigCSRre", "MHC", "BRAT-BW", "COMPASSS", "DDBJ", "Olelo", "mGrid", "c-REDUCE", "PReMod", "POINeT", "NoiseMaker", "MASTR", "PINE-SPARKY", "HTHquery", "PeakLink", "QualitySNPng", "miR-EdiTar", "FISH", "MethyQA", "QuasiMotiFinder", "ntCard", "RNATOPS-W", "MMAD", "Cyclone", "ClinVar", "5-methylcytosine in RNA", "REDCap", "CBrowse", "CodingMotif", "GREAT", "CoFactor", "BIPAD", "RAPID", "DESM", "WebGL", "CerealsDB", "LOT", "SROOGLE", "bc-GenExMiner 3.0", "MuD", "POSTAR", "RNALocate", "PileLineGUI", "SCAN", "nextflu", "regioneR", "APID", "g", "KFC", "CellWhere", "omiRas", "GtRNAdb", "EuGene", "GeneEvolve", "Seqinspector", "NMRFAM-SPARKY", "Multi-CAR", "SIREs", "OpWise", "LBSizeCleav", "NOMAD-Ref", "GenomeRNAi", "MQAPRank", "SALAMI", "svapls", "CyTRANSFINDER", "ERIC", "FORCE", "TRbase", "COEXPEDIA", "FORTE", "MATLAB-based", "ProDy", "TaxonGap", "Full-Malaria/Parasites and Full-Arthropods", "FoldAtlas", "BABELOMICS", "DeepBlue", "XXmotif", "IView", "GraP", "WebArrayDB", "STEME", "RAPHAEL", "SBML2LaTEX", "GASH", "GCORE-sib", "GEM", "MobiDB", "PFP", "SALIGN", "IBRENA", "d-Omix", "SEAL", "leeHom", "mSpecs", "TiGER", "TherMos", "SwissDock", "IonGAP", "RF-Cloning.org", "BiGG", "KEGGtranslator", "PGDSpider", "MCMC-ODPR", "CentiBiN", "MCPep", "FASTdoop", "MITSU", "Automics", "RiceWiki", "LC-IMS-MS Feature Finder", "ECL", "SEQADAPT", "JuncDB", "PyMod", "GENIES", "SEPPA", "iPDA", "IDEA", "DisProt", "SAGE", "Bellerophon", "Gobe", "NOBLAST", "GeneSCF", "LDlink", "WormNet", "NCI", "SwissRegulon", "SensiPath", "MORPHIN", "WebPSN", "htsint", "HPG", "COMPARE", "PepCrawler", "RepExplore", "LIMS", "tranSMART-XNAT", "TriTrypDB", "HBVdb", "SVM-HUSTLE\u2014an", "PDID", "GeneWeaver", "ProDom", "lnCaNet", "SINA", "Recode-2", "RNA-SeQC", "MobiDB-lite", "SpliceJumper", "OSA", "DIANA-mirPath", "piPipes", "COLOMBOS", "FRASS", "dAUTObase", "CAVER", "Chromhome", "LocusZoom", "MoMA-LigPath", "ZCURVE", "fRMA", "CAFE", "GOLD", "CCDS", "MAGNA", "EnrichNet", "diCal-IBD", "Babelomics", "BTR", "Bhageerath", "ProBiS-ligands", "iGTP", "PatternCNV", "PS2-v2", "SNPMaP", "FTFlex", "CATMA", "HaploReg", "CNB-MAC", "AMMOS2", "SIDEKICK", "AcCNET", "R-Syst", "BioBloom", "MaConDa", "miRMaid", "taveRNA", "tbvar", "MISTIC", "BioCause", "TM4", "HandAlign", "StrainInfo", "WebGMAP", "MotifMap", "SimRare", "BindUP", "BeAtMuSiC", "array-CGH", "KNIME", "MiRTif", "ZInC", "BioLiP", "Frog2", "MPAgenomics", "MaxLink", "YMDB", "BioNetGen", "RegLRSD", "TreeDom", "BIOADI", "TOPCONS", "SyMAP", "Jetset", "3D", "PartsList", "synbreed", "ConSeq", "ENCODE", "pVOGs", "CircaDB", "MultiDataSet", "CloneCNA", "AnkPlex", "MyHits", "PHUSER", "GREVE", "TogoTable", "\"\"\"PolyMin\"\"\"", "deltaGseg", "PIntron", "LS-SNP/PDB", "SVM-RFE", "flowVS", "MARAN", "Caleydo", "ThreaDomEx", "CLIPZ", "SEQuel", "mycoCLAP", "MRS", "miRcode", "HLA-check", "GRiP", "LibSBMLSim", "SeqDepot", "SignalViewer", "OrthoNets", "Hydra-Multi", "DB-AT", "CicArVarDB", "MODOMICS", "RNAseqViewer", "ASTRAL", "LNCipedia", "CABS-dock", "ODTbrain", "ExpaRNA-P", "scrm", "ModLoop", "Gene3D", "PhosphoPICK", "MetaFast", "forqs", "Mcheza", "Argot2", "COSMOS", "Lists2Networks", "NNAlign", "PlantGDB", "Cancer3D", "ProCMD", "MetaPathways", "FACETS", "MyMpn", "Vespucci", "NCG", "CABeRNET", "COG", "OrthoDB", "TSpred", "BSMAP", "MeV", "L1Base", "WERAM", "QTLMiner", "HoPaCI-DB", "IDR", "Parseq", "FlyTF", "VIZ-GRAIL", "SCLD", "ProbOnto", "LASSIE", "TRIg", "Mireval", "NFFinder", "MeltDB", "HeatMapper", "ProQDock", "PRIN", "EasyStrata", "W.A.T.E.R.S.", "CLAST", "GEPdb", "JEnsembl", "T-REX", "ABS", "Polbase", "YY1TargetDB", "ChromDB", "UniProt", "ENCoM", "AptRank", "GiRaF", "PIP", "Reveel", "GlycoDigest", "IC4R", "Murlet", "solGS", "PanGEA", "PLEXY", "CCHMM_PROF", "SMARTS", "TreeQTL", "HMM-ModE", "Twine", "FASTQ", "DRAGON", "fmcsR", "mDCC_tools", "ABACAS", "PolySearch2", "GoArrays", "RESQUE", "SSWAP", "HIV-1 human interaction database", "TRAL", "libfbi", "GOTA", "caRpools", "FunFOLD", "CRISPRs", "PeptideMine", "DupChecker", "DNA-BAR", "miRPath", "RNApredator", "ToxReporter", "ECplot", "DISSCO", "pE-DB", "siVirus", "CIPRO", "MICC", "MedlineRanker", "Pathway-PDT", "trimAl", "SwissLipids", "MetSizeR", "CancerGenes", "iAnn", "KMC", "GUIDANCE2", "cit", "RVD2", "CSCdb", "NGS-eval", "NetComm", "DiRE", "ArrayXPath", "VEGAWES", "Rknots", "BioNetFit", "pepMMsMIMIC", "Nemo", "FiberDock", "BLAST2GENE", "e2g", "morFeus", "ChEMBL", "FuGePrior", "Gemma", "SAbDab", "BioFVM", "CrisprGE", "RENi", "Ruffus", "Epiviz", "DIST", "ZFIN", "ChromA", "TAL", "MDWeb", "PrimerHunter", "BrEPS", "CMGSDB", "ABySS", "multi-GPU", "PEASE", "CDSfold", "readat", "NOBAI", "CauloBrowser", "GORpipe", "APTANI", "ClusterMine360", "DBBP", "SARA-Coffee", "OMIA", "UbiNet", "ProbCD", "MGEScan-non-LTR", "NPIDB", "PIDO", "WoPPER", "seXY", "STRIDE", "ESS++", "MEME", "GOEAST", "w4CSeq", "P-MITE", "msTALI", "WinHAP2", "PLAN2L", "GrammR", "mirPub", "SODA", "TaggerOne", "FDA", "ONDEX", "Islander", "SCNVSim", "OMBlast", "DEG", "GSDS", "LBIBCell", "Transducers", "PISCES", "IBD", "myExperiment", "B2G-FAR", "EMDataBank", "SitePainter", "EBARDenovo", "MSIsensor", "BioQuery", "AlloPred", "PDBTM", "PileLine", "ASSIsT", "HGT-DB", "Snpdat", "ALE-HSA21", "CoRAL", "ModFOLD6", "BIGMAC", "SIMCOAL", "nCal", "iGEMDOCK", "REJECTOR", "Bionemo", "PROTMAP2D", "ECMDB", "CoGAPS", "BioMercator", "SCANMS", "RONN", "Target-align", "pepATTRACT", "DynaMine", "ADAPT-NMR Enhancer", "PDB", "adLIMS", "SDM", "GOSemSim", "PeroxiBase", "TCMID", "RRTree", "SPIDer", "PaPI", "VSGdb", "ProfCom", "MeQA", "dsCheck", "pyDockSAXS", "Web-Beagle", "CEAS", "CREx", "PathwaysWeb", "ABMapper", "ccPDB", "PREX", "SCOWLP", "ODB", "Reads2Type", "ProtPhylo", "SPoRE", "MIPModDB", "SwissVar", "WGCNA", "COLORADO3D", "MIG", "FunctSNP", "PlantGSEA", "ICoVeR", "MaxAlign", "CAGExploreR", "maxAlike", "ModuleOrganizer", "TFBScluster", "MS-REDUCE", "proGenomes", "R.E.D.", "miRanalyzer", "GOrilla", "FlaiMapper", "PmiRKB", "RING", "DEXUS", "e-Driver", "miRU", "HMMConverter1.0", "MAAMD", "MSAcquisitionSimulator", "SMILES-based", "VARiD", "SHERPA", "JAWAMix5", "ESG", "GAP", "PMDBase", "DIALIGN", "NPACT", "CADRE", "PGA", "Circles", "MultiBind", "SIMPLE", "Malin", "cooccurNet", "ExaML", "Pharmit", "BadiRate", "HGNC", "Metal-MACiE", "PINCAGE", "OnLine", "GenoList", "Ergatis", "RNAfbinv", "PAT", "GAM-NGS", "CAMUR", "SCOPPI", "DMDM", "JBEI", "Orphelia", "flowDensity", "HMM-based", "Jimena", "Fizzy", "genoPlotR", "NetworkBLAST", "ExactFDR", "DGET", "ArachnoServer", "T2DM-GeneMiner", "VegaMC", "SilkSatDb", "PepPSy", "RAIN", "RRE", "Xdigitise", "PAWE-3D", "Discoal", "CorreLogo", "Sambamba", "OriDB", "PHYLUCE", "PlantCAZyme", "SmedGD", "GraphWeb", "RNAex", "PRECOG", "PALMA", "PolyDoms", "Methyl-Analyzer\u2014whole", "NGC", "MASQOT-GUI", "PPI3D", "Gimli", "PAGED", "iProClass", "PCRPi-DB", "RCD+", "POCO", "ChromoWheel", "Xenbase", "ROCR", "Egas", "NMR-based", "CHOPIN", "SUSPECTS", "CplexA", "CARLSBAD", "PepCyber", "BCov", "PIR", "FlyRNAi", "MOCAT2", "Spial", "ARB", "SelenoDB", "COMBREX", "IGDD", "FluTyper-an", "GenClust", "CRISPRz", "bioDBnet", "PyMS", "MultiAlign", "Colonyzer", "JEPEG", "SBaddon", "FTEC", "VisualGraphX", "DeNoGAP", "STELLS2", "ITM", "CodonLogo", "AluHunter", "GoMapMan", "BMRF-Net", "SomamiR"], "weights": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.184564907385, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.24832969684, 0.0, 0.0, 0.0, 0.0, 0.460745376075, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.24832969684, 0.0, 0.0, 0.0, 0.460745376075, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.289115194016, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.166024663743, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204347111472, 0.0, 0.16364167163, 0.16364167163, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.289115194016, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.226927607774, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.318715871505, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.166024663743, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204347111472, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.226927607774, 0.0, 0.0, 0.0, 0.0, 0.0, 0.318715871505, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.242421929991, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.244221290753, 0.244221290753, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.242421929991, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.116764032634, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.343984784548, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.192617421268, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.116764032634, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.343984784548, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.202092000615, 0.202092000615, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.192617421268, 0.0, 0.0, 0.15209182397, 0.15209182397, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.188499808997, 0.188499808997, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.371919794313, 0.0, 0.0, 0.0, 0.0, 0.0, 0.373938873648, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.284198802176, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.404004685697, 0.404004685697, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.371919794313, 0.0, 0.0, 0.0, 0.373938873648, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.284198802176, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.33844479791, 0.33844479791, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.134094619875, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.134094619875, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14046766333, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.14046766333, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.270513542992, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.210573455086, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.270513542992, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.210573455086, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.276537413326, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.181377053524, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.181377053524, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.276537413326, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.209346990966, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.192053146704, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.284826775731, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.195547890334, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.209346990966, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.285927503045, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.192053146704, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.284826775731, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.195547890334, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.285927503045, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.391765657045, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.339213138467, 0.0, 0.318358011652, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.224675893043, 0.224675893043, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.463572741378, 0.463572741378, 0.0, 0.0, 0.391765657045, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.214519464412, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.339213138467, 0.0, 0.318358011652, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.161283971541, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.214519464412, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.161283971541, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.363042185234, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.363042185234, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.556941131723, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.556941131723, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.634814792276, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.634814792276, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.219386389551, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.268404776385, 0.268404776385, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.219386389551, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.189317531095, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.49412283508, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.189317531095, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.49412283508, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.1459886339, 0.1459886339, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.299859806294, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.272548035187, 0.272548035187, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.299859806294, 0.0, 0.0, 0.0, 0.0, 0.0, 0.271511159874, 0.271511159874, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.401747019999, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0866355901521, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.401747019999, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0866355901521, 0.0, 0.0, 0.0, 0.0, 0.400132668242, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.294046873344, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.205372937078, 0.0, 0.0, 0.0, 0.0, 0.0, 0.400132668242, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.245060163838, 0.245060163838, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.294046873344, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.205372937078, 0.0, 0.0, 0.138723287632, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.330073791274, 0.21116347668, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.186079750745, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.138723287632, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.330073791274, 0.21116347668, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.186079750745, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.358094087498, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204561313361, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.358094087498, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.135426283, 0.135426283, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204561313361, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.15619217385, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.15619217385, 0.0, 0.246174046429, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.252349051044, 0.0, 0.0, 0.246174046429, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.230739728015, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.346652118975, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.252349051044, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.230739728015, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.346652118975, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.164406163813, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.164406163813, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.216924517957, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.364914670202, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.381578288193, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.216533429318, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.216924517957, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204356047474, 0.0, 0.0, 0.0, 0.0, 0.0, 0.364914670202, 0.0, 0.313186609365, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.381578288193, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.216533429318, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.204356047474, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.313186609365, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.134137645235, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.134137645235, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.251004675268, 0.251004675268, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.15738553676, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.164946696524, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.397128232273, 0.397128232273, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.15738553676, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.164946696524, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.235845351321, 0.235845351321, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.183717238334, 0.183717238334, 0.0, 0.0, 0.0, 0.168448770325, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.21063420553, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.168448770325, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.21063420553, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.184915631813, 0.184915631813, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.158136057364, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.385889295243, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.158136057364, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.385889295243, 0.0, 0.0, 0.0, 0.0, 0.306317238241, 0.306317238241, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.274173386346, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.278860647344, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.274173386346, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.278860647344, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.209889551505, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.47860555086, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.184564907385, 1.0, 1.0, 0.0, 0.0, 0.268715999355, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]};

var infos = {};

d3.select('textarea#genes').attr('disabled', 'true').text('No GMT File Inputted');

d3.select('input#submitButton').attr('disabled', 'true');

d3.select('input#exampleButton').attr('disabled', 'true');





	getJSON(json, [0, 255, 255], [255, 255, 255]);

	if (typeof infos != "undefined") {
		G_VAR.infos = infos;
	}


	storeFact = [0,0];



	d3.select("#holdSVG").append("div")
			.attr("id","infoReminder")
			.style("width", "390px").style("padding-top", "15px")
			.text("Click a spot on the canvas to view more information about the term.");

	if (isEmpty(G_VAR.infos))
	{
		d3.select("#main").on("mousedown", null);
		d3.select("#main").on("mousedown", declareName);
		d3.select("#tabFind2").attr("onclick", null).text("");
	}

</script>

</html>